2018-09-02 16:48:19,509;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 16:48:19,510;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT VERSION(); args=None
2018-09-02 16:48:19,543;DEBUG;utils.execute;Line:91;Message:(0.003) SHOW FULL TABLES; args=None
2018-09-02 16:48:19,546;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2018-09-02 16:50:06,793;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 16:50:06,795;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 16:50:06,825;DEBUG;utils.execute;Line:91;Message:(0.003) SHOW FULL TABLES; args=None
2018-09-02 16:50:06,826;DEBUG;schema.execute;Line:110;Message:CREATE TABLE `django_migrations` (`id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY, `app` varchar(255) NOT NULL, `name` varchar(255) NOT NULL, `applied` datetime(6) NOT NULL); (params None)
2018-09-02 16:50:06,973;DEBUG;utils.execute;Line:91;Message:(0.146) CREATE TABLE `django_migrations` (`id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY, `app` varchar(255) NOT NULL, `name` varchar(255) NOT NULL, `applied` datetime(6) NOT NULL); args=None
2018-09-02 16:50:06,976;DEBUG;utils.execute;Line:91;Message:(0.002) SELECT engine FROM information_schema.tables WHERE table_name = 'django_migrations'; args=[u'django_migrations']
2018-09-02 16:50:06,976;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'django_migrations'; args=[u'django_migrations']
2018-09-02 16:50:06,977;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'django_migrations'; args=[u'django_migrations']
2018-09-02 16:50:06,979;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'django_migrations'; args=[u'django_migrations']
2018-09-02 16:50:06,983;DEBUG;utils.execute;Line:91;Message:(0.003) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2018-09-02 16:50:22,463;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 16:50:22,463;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@sql_mode; args=None
2018-09-02 16:50:22,464;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 16:50:22,489;DEBUG;utils.execute;Line:91;Message:(0.002) SHOW FULL TABLES; args=None
2018-09-02 16:50:22,493;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2018-09-02 16:50:22,493;DEBUG;utils.execute;Line:91;Message:(0.001) SHOW FULL TABLES; args=None
2018-09-02 16:50:22,496;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2018-09-02 16:50:22,503;DEBUG;schema.execute;Line:110;Message:CREATE TABLE `django_content_type` (`id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY, `name` varchar(100) NOT NULL, `app_label` varchar(100) NOT NULL, `model` varchar(100) NOT NULL); (params None)
2018-09-02 16:50:22,674;DEBUG;utils.execute;Line:91;Message:(0.171) CREATE TABLE `django_content_type` (`id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY, `name` varchar(100) NOT NULL, `app_label` varchar(100) NOT NULL, `model` varchar(100) NOT NULL); args=None
2018-09-02 16:50:22,676;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'django_content_type'; args=[u'django_content_type']
2018-09-02 16:50:22,677;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'django_content_type'; args=[u'django_content_type']
2018-09-02 16:50:22,677;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT engine FROM information_schema.tables WHERE table_name = 'django_content_type'; args=[u'django_content_type']
2018-09-02 16:50:22,678;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT engine FROM information_schema.tables WHERE table_name = 'django_content_type'; args=[u'django_content_type']
2018-09-02 16:50:22,680;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `django_content_type` ADD CONSTRAINT `django_content_type_app_label_model_76bd3d3b_uniq` UNIQUE (`app_label`, `model`); (params ())
2018-09-02 16:50:22,700;DEBUG;utils.execute;Line:91;Message:(0.018) ALTER TABLE `django_content_type` ADD CONSTRAINT `django_content_type_app_label_model_76bd3d3b_uniq` UNIQUE (`app_label`, `model`); args=()
2018-09-02 16:50:22,701;DEBUG;utils.execute;Line:91;Message:(0.001) SHOW FULL TABLES; args=None
2018-09-02 16:50:22,703;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `django_migrations` (`app`, `name`, `applied`) VALUES ('contenttypes', '0001_initial', '2018-09-02 04:50:22.702000'); args=['contenttypes', u'0001_initial', u'2018-09-02 04:50:22.702000']
2018-09-02 16:50:22,707;DEBUG;schema.execute;Line:110;Message:CREATE TABLE `auth_permission` (`id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY, `name` varchar(50) NOT NULL, `content_type_id` integer NOT NULL, `codename` varchar(100) NOT NULL); (params None)
2018-09-02 16:50:22,765;DEBUG;utils.execute;Line:91;Message:(0.057) CREATE TABLE `auth_permission` (`id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY, `name` varchar(50) NOT NULL, `content_type_id` integer NOT NULL, `codename` varchar(100) NOT NULL); args=None
2018-09-02 16:50:22,766;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_permission'; args=[u'auth_permission']
2018-09-02 16:50:22,767;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_permission'; args=[u'auth_permission']
2018-09-02 16:50:22,769;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_permission'; args=[u'auth_permission']
2018-09-02 16:50:22,769;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_permission'; args=[u'auth_permission']
2018-09-02 16:50:22,773;DEBUG;schema.execute;Line:110;Message:CREATE TABLE `auth_group` (`id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY, `name` varchar(80) NOT NULL UNIQUE); (params None)
2018-09-02 16:50:22,831;DEBUG;utils.execute;Line:91;Message:(0.055) CREATE TABLE `auth_group` (`id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY, `name` varchar(80) NOT NULL UNIQUE); args=None
2018-09-02 16:50:22,832;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_group'; args=[u'auth_group']
2018-09-02 16:50:22,834;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_group'; args=[u'auth_group']
2018-09-02 16:50:22,834;DEBUG;schema.execute;Line:110;Message:CREATE TABLE `auth_group_permissions` (`id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY, `group_id` integer NOT NULL, `permission_id` integer NOT NULL); (params None)
2018-09-02 16:50:22,878;DEBUG;utils.execute;Line:91;Message:(0.044) CREATE TABLE `auth_group_permissions` (`id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY, `group_id` integer NOT NULL, `permission_id` integer NOT NULL); args=None
2018-09-02 16:50:22,881;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_group_permissions'; args=[u'auth_group_permissions']
2018-09-02 16:50:22,881;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_group_permissions'; args=[u'auth_group_permissions']
2018-09-02 16:50:22,882;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_group_permissions'; args=[u'auth_group_permissions']
2018-09-02 16:50:22,891;DEBUG;schema.execute;Line:110;Message:CREATE TABLE `auth_user` (`id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY, `password` varchar(128) NOT NULL, `last_login` datetime(6) NOT NULL, `is_superuser` bool NOT NULL, `username` varchar(30) NOT NULL UNIQUE, `first_name` varchar(30) NOT NULL, `last_name` varchar(30) NOT NULL, `email` varchar(75) NOT NULL, `is_staff` bool NOT NULL, `is_active` bool NOT NULL, `date_joined` datetime(6) NOT NULL); (params None)
2018-09-02 16:50:22,937;DEBUG;utils.execute;Line:91;Message:(0.045) CREATE TABLE `auth_user` (`id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY, `password` varchar(128) NOT NULL, `last_login` datetime(6) NOT NULL, `is_superuser` bool NOT NULL, `username` varchar(30) NOT NULL UNIQUE, `first_name` varchar(30) NOT NULL, `last_name` varchar(30) NOT NULL, `email` varchar(75) NOT NULL, `is_staff` bool NOT NULL, `is_active` bool NOT NULL, `date_joined` datetime(6) NOT NULL); args=None
2018-09-02 16:50:22,937;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_user'; args=[u'auth_user']
2018-09-02 16:50:22,938;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_user'; args=[u'auth_user']
2018-09-02 16:50:22,940;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_user'; args=[u'auth_user']
2018-09-02 16:50:22,941;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_user'; args=[u'auth_user']
2018-09-02 16:50:22,943;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_user'; args=[u'auth_user']
2018-09-02 16:50:22,944;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_user'; args=[u'auth_user']
2018-09-02 16:50:22,944;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_user'; args=[u'auth_user']
2018-09-02 16:50:22,946;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_user'; args=[u'auth_user']
2018-09-02 16:50:22,950;DEBUG;utils.execute;Line:91;Message:(0.002) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_user'; args=[u'auth_user']
2018-09-02 16:50:22,950;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_user'; args=[u'auth_user']
2018-09-02 16:50:22,951;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_user'; args=[u'auth_user']
2018-09-02 16:50:22,953;DEBUG;schema.execute;Line:110;Message:CREATE TABLE `auth_user_groups` (`id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY, `user_id` integer NOT NULL, `group_id` integer NOT NULL); (params None)
2018-09-02 16:50:22,990;DEBUG;utils.execute;Line:91;Message:(0.038) CREATE TABLE `auth_user_groups` (`id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY, `user_id` integer NOT NULL, `group_id` integer NOT NULL); args=None
2018-09-02 16:50:22,992;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_user_groups'; args=[u'auth_user_groups']
2018-09-02 16:50:22,993;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_user_groups'; args=[u'auth_user_groups']
2018-09-02 16:50:22,993;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_user_groups'; args=[u'auth_user_groups']
2018-09-02 16:50:22,994;DEBUG;schema.execute;Line:110;Message:CREATE TABLE `auth_user_user_permissions` (`id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY, `user_id` integer NOT NULL, `permission_id` integer NOT NULL); (params None)
2018-09-02 16:50:23,029;DEBUG;utils.execute;Line:91;Message:(0.034) CREATE TABLE `auth_user_user_permissions` (`id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY, `user_id` integer NOT NULL, `permission_id` integer NOT NULL); args=None
2018-09-02 16:50:23,030;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_user_user_permissions'; args=[u'auth_user_user_permissions']
2018-09-02 16:50:23,032;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_user_user_permissions'; args=[u'auth_user_user_permissions']
2018-09-02 16:50:23,033;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_user_user_permissions'; args=[u'auth_user_user_permissions']
2018-09-02 16:50:23,033;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `auth_permission` ADD CONSTRAINT `auth_permission_content_type_id_2f476e4b_fk_django_co` FOREIGN KEY (`content_type_id`) REFERENCES `django_content_type` (`id`); (params ())
2018-09-02 16:50:23,171;DEBUG;utils.execute;Line:91;Message:(0.138) ALTER TABLE `auth_permission` ADD CONSTRAINT `auth_permission_content_type_id_2f476e4b_fk_django_co` FOREIGN KEY (`content_type_id`) REFERENCES `django_content_type` (`id`); args=()
2018-09-02 16:50:23,171;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `auth_permission` ADD CONSTRAINT `auth_permission_content_type_id_codename_01ab375a_uniq` UNIQUE (`content_type_id`, `codename`); (params ())
2018-09-02 16:50:23,204;DEBUG;utils.execute;Line:91;Message:(0.032) ALTER TABLE `auth_permission` ADD CONSTRAINT `auth_permission_content_type_id_codename_01ab375a_uniq` UNIQUE (`content_type_id`, `codename`); args=()
2018-09-02 16:50:23,204;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `auth_group_permissions` ADD CONSTRAINT `auth_group_permissions_group_id_b120cbf9_fk_auth_group_id` FOREIGN KEY (`group_id`) REFERENCES `auth_group` (`id`); (params ())
2018-09-02 16:50:23,298;DEBUG;utils.execute;Line:91;Message:(0.093) ALTER TABLE `auth_group_permissions` ADD CONSTRAINT `auth_group_permissions_group_id_b120cbf9_fk_auth_group_id` FOREIGN KEY (`group_id`) REFERENCES `auth_group` (`id`); args=()
2018-09-02 16:50:23,298;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `auth_group_permissions` ADD CONSTRAINT `auth_group_permissio_permission_id_84c5c92e_fk_auth_perm` FOREIGN KEY (`permission_id`) REFERENCES `auth_permission` (`id`); (params ())
2018-09-02 16:50:23,381;DEBUG;utils.execute;Line:91;Message:(0.082) ALTER TABLE `auth_group_permissions` ADD CONSTRAINT `auth_group_permissio_permission_id_84c5c92e_fk_auth_perm` FOREIGN KEY (`permission_id`) REFERENCES `auth_permission` (`id`); args=()
2018-09-02 16:50:23,381;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `auth_group_permissions` ADD CONSTRAINT `auth_group_permissions_group_id_permission_id_0cd325b0_uniq` UNIQUE (`group_id`, `permission_id`); (params ())
2018-09-02 16:50:23,404;DEBUG;utils.execute;Line:91;Message:(0.024) ALTER TABLE `auth_group_permissions` ADD CONSTRAINT `auth_group_permissions_group_id_permission_id_0cd325b0_uniq` UNIQUE (`group_id`, `permission_id`); args=()
2018-09-02 16:50:23,404;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `auth_user_groups` ADD CONSTRAINT `auth_user_groups_user_id_6a12ed8b_fk_auth_user_id` FOREIGN KEY (`user_id`) REFERENCES `auth_user` (`id`); (params ())
2018-09-02 16:50:23,642;DEBUG;utils.execute;Line:91;Message:(0.236) ALTER TABLE `auth_user_groups` ADD CONSTRAINT `auth_user_groups_user_id_6a12ed8b_fk_auth_user_id` FOREIGN KEY (`user_id`) REFERENCES `auth_user` (`id`); args=()
2018-09-02 16:50:23,644;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `auth_user_groups` ADD CONSTRAINT `auth_user_groups_group_id_97559544_fk_auth_group_id` FOREIGN KEY (`group_id`) REFERENCES `auth_group` (`id`); (params ())
2018-09-02 16:50:23,734;DEBUG;utils.execute;Line:91;Message:(0.091) ALTER TABLE `auth_user_groups` ADD CONSTRAINT `auth_user_groups_group_id_97559544_fk_auth_group_id` FOREIGN KEY (`group_id`) REFERENCES `auth_group` (`id`); args=()
2018-09-02 16:50:23,734;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `auth_user_groups` ADD CONSTRAINT `auth_user_groups_user_id_group_id_94350c0c_uniq` UNIQUE (`user_id`, `group_id`); (params ())
2018-09-02 16:50:23,763;DEBUG;utils.execute;Line:91;Message:(0.028) ALTER TABLE `auth_user_groups` ADD CONSTRAINT `auth_user_groups_user_id_group_id_94350c0c_uniq` UNIQUE (`user_id`, `group_id`); args=()
2018-09-02 16:50:23,765;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `auth_user_user_permissions` ADD CONSTRAINT `auth_user_user_permissions_user_id_a95ead1b_fk_auth_user_id` FOREIGN KEY (`user_id`) REFERENCES `auth_user` (`id`); (params ())
2018-09-02 16:50:23,868;DEBUG;utils.execute;Line:91;Message:(0.104) ALTER TABLE `auth_user_user_permissions` ADD CONSTRAINT `auth_user_user_permissions_user_id_a95ead1b_fk_auth_user_id` FOREIGN KEY (`user_id`) REFERENCES `auth_user` (`id`); args=()
2018-09-02 16:50:23,868;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `auth_user_user_permissions` ADD CONSTRAINT `auth_user_user_permi_permission_id_1fbb5f2c_fk_auth_perm` FOREIGN KEY (`permission_id`) REFERENCES `auth_permission` (`id`); (params ())
2018-09-02 16:50:23,967;DEBUG;utils.execute;Line:91;Message:(0.097) ALTER TABLE `auth_user_user_permissions` ADD CONSTRAINT `auth_user_user_permi_permission_id_1fbb5f2c_fk_auth_perm` FOREIGN KEY (`permission_id`) REFERENCES `auth_permission` (`id`); args=()
2018-09-02 16:50:23,967;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `auth_user_user_permissions` ADD CONSTRAINT `auth_user_user_permissions_user_id_permission_id_14a6b632_uniq` UNIQUE (`user_id`, `permission_id`); (params ())
2018-09-02 16:50:23,980;DEBUG;utils.execute;Line:91;Message:(0.012) ALTER TABLE `auth_user_user_permissions` ADD CONSTRAINT `auth_user_user_permissions_user_id_permission_id_14a6b632_uniq` UNIQUE (`user_id`, `permission_id`); args=()
2018-09-02 16:50:23,983;DEBUG;utils.execute;Line:91;Message:(0.003) SHOW FULL TABLES; args=None
2018-09-02 16:50:23,984;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `django_migrations` (`app`, `name`, `applied`) VALUES ('auth', '0001_initial', '2018-09-02 04:50:23.983000'); args=['auth', u'0001_initial', u'2018-09-02 04:50:23.983000']
2018-09-02 16:50:23,999;DEBUG;schema.execute;Line:110;Message:CREATE TABLE `django_admin_log` (`id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY, `action_time` datetime(6) NOT NULL, `object_id` longtext NULL, `object_repr` varchar(200) NOT NULL, `action_flag` smallint UNSIGNED NOT NULL, `change_message` longtext NOT NULL, `content_type_id` integer NULL, `user_id` integer NOT NULL); (params None)
2018-09-02 16:50:24,036;DEBUG;utils.execute;Line:91;Message:(0.038) CREATE TABLE `django_admin_log` (`id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY, `action_time` datetime(6) NOT NULL, `object_id` longtext NULL, `object_repr` varchar(200) NOT NULL, `action_flag` smallint UNSIGNED NOT NULL, `change_message` longtext NOT NULL, `content_type_id` integer NULL, `user_id` integer NOT NULL); args=None
2018-09-02 16:50:24,038;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'django_admin_log'; args=[u'django_admin_log']
2018-09-02 16:50:24,039;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'django_admin_log'; args=[u'django_admin_log']
2018-09-02 16:50:24,040;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'django_admin_log'; args=[u'django_admin_log']
2018-09-02 16:50:24,042;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'django_admin_log'; args=[u'django_admin_log']
2018-09-02 16:50:24,042;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'django_admin_log'; args=[u'django_admin_log']
2018-09-02 16:50:24,043;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'django_admin_log'; args=[u'django_admin_log']
2018-09-02 16:50:24,045;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'django_admin_log'; args=[u'django_admin_log']
2018-09-02 16:50:24,046;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT engine FROM information_schema.tables WHERE table_name = 'django_admin_log'; args=[u'django_admin_log']
2018-09-02 16:50:24,048;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `django_admin_log` ADD CONSTRAINT `django_admin_log_content_type_id_c4bce8eb_fk_django_co` FOREIGN KEY (`content_type_id`) REFERENCES `django_content_type` (`id`); (params ())
2018-09-02 16:50:24,141;DEBUG;utils.execute;Line:91;Message:(0.092) ALTER TABLE `django_admin_log` ADD CONSTRAINT `django_admin_log_content_type_id_c4bce8eb_fk_django_co` FOREIGN KEY (`content_type_id`) REFERENCES `django_content_type` (`id`); args=()
2018-09-02 16:50:24,141;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `django_admin_log` ADD CONSTRAINT `django_admin_log_user_id_c564eba6_fk_auth_user_id` FOREIGN KEY (`user_id`) REFERENCES `auth_user` (`id`); (params ())
2018-09-02 16:50:24,213;DEBUG;utils.execute;Line:91;Message:(0.072) ALTER TABLE `django_admin_log` ADD CONSTRAINT `django_admin_log_user_id_c564eba6_fk_auth_user_id` FOREIGN KEY (`user_id`) REFERENCES `auth_user` (`id`); args=()
2018-09-02 16:50:24,216;DEBUG;utils.execute;Line:91;Message:(0.001) SHOW FULL TABLES; args=None
2018-09-02 16:50:24,217;DEBUG;utils.execute;Line:91;Message:(0.000) INSERT INTO `django_migrations` (`app`, `name`, `applied`) VALUES ('admin', '0001_initial', '2018-09-02 04:50:24.216000'); args=['admin', u'0001_initial', u'2018-09-02 04:50:24.216000']
2018-09-02 16:50:24,229;DEBUG;utils.execute;Line:91;Message:(0.001) SHOW FULL TABLES; args=None
2018-09-02 16:50:24,230;DEBUG;utils.execute;Line:91;Message:(0.000) INSERT INTO `django_migrations` (`app`, `name`, `applied`) VALUES ('admin', '0002_logentry_remove_auto_add', '2018-09-02 04:50:24.229000'); args=['admin', u'0002_logentry_remove_auto_add', u'2018-09-02 04:50:24.229000']
2018-09-02 16:50:24,240;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `django_content_type` MODIFY `name` varchar(100) NULL; (params [])
2018-09-02 16:50:24,334;DEBUG;utils.execute;Line:91;Message:(0.092) ALTER TABLE `django_content_type` MODIFY `name` varchar(100) NULL; args=[]
2018-09-02 16:50:24,345;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `django_content_type` DROP COLUMN `name`; (params ())
2018-09-02 16:50:24,394;DEBUG;utils.execute;Line:91;Message:(0.048) ALTER TABLE `django_content_type` DROP COLUMN `name`; args=()
2018-09-02 16:50:24,395;DEBUG;utils.execute;Line:91;Message:(0.001) SHOW FULL TABLES; args=None
2018-09-02 16:50:24,398;DEBUG;utils.execute;Line:91;Message:(0.000) INSERT INTO `django_migrations` (`app`, `name`, `applied`) VALUES ('contenttypes', '0002_remove_content_type_name', '2018-09-02 04:50:24.397000'); args=['contenttypes', u'0002_remove_content_type_name', u'2018-09-02 04:50:24.397000']
2018-09-02 16:50:24,404;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `auth_permission` MODIFY `name` varchar(255) NOT NULL; (params [])
2018-09-02 16:50:24,618;DEBUG;utils.execute;Line:91;Message:(0.214) ALTER TABLE `auth_permission` MODIFY `name` varchar(255) NOT NULL; args=[]
2018-09-02 16:50:24,621;DEBUG;utils.execute;Line:91;Message:(0.001) SHOW FULL TABLES; args=None
2018-09-02 16:50:24,621;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `django_migrations` (`app`, `name`, `applied`) VALUES ('auth', '0002_alter_permission_name_max_length', '2018-09-02 04:50:24.621000'); args=['auth', u'0002_alter_permission_name_max_length', u'2018-09-02 04:50:24.621000']
2018-09-02 16:50:24,632;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `auth_user` MODIFY `email` varchar(254) NOT NULL; (params [])
2018-09-02 16:50:24,729;DEBUG;utils.execute;Line:91;Message:(0.096) ALTER TABLE `auth_user` MODIFY `email` varchar(254) NOT NULL; args=[]
2018-09-02 16:50:24,730;DEBUG;utils.execute;Line:91;Message:(0.001) SHOW FULL TABLES; args=None
2018-09-02 16:50:24,732;DEBUG;utils.execute;Line:91;Message:(0.000) INSERT INTO `django_migrations` (`app`, `name`, `applied`) VALUES ('auth', '0003_alter_user_email_max_length', '2018-09-02 04:50:24.731000'); args=['auth', u'0003_alter_user_email_max_length', u'2018-09-02 04:50:24.731000']
2018-09-02 16:50:24,743;DEBUG;utils.execute;Line:91;Message:(0.001) SHOW FULL TABLES; args=None
2018-09-02 16:50:24,744;DEBUG;utils.execute;Line:91;Message:(0.000) INSERT INTO `django_migrations` (`app`, `name`, `applied`) VALUES ('auth', '0004_alter_user_username_opts', '2018-09-02 04:50:24.743000'); args=['auth', u'0004_alter_user_username_opts', u'2018-09-02 04:50:24.743000']
2018-09-02 16:50:24,756;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `auth_user` MODIFY `last_login` datetime(6) NULL; (params [])
2018-09-02 16:50:24,809;DEBUG;utils.execute;Line:91;Message:(0.053) ALTER TABLE `auth_user` MODIFY `last_login` datetime(6) NULL; args=[]
2018-09-02 16:50:24,809;DEBUG;utils.execute;Line:91;Message:(0.001) SHOW FULL TABLES; args=None
2018-09-02 16:50:24,812;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `django_migrations` (`app`, `name`, `applied`) VALUES ('auth', '0005_alter_user_last_login_null', '2018-09-02 04:50:24.811000'); args=['auth', u'0005_alter_user_last_login_null', u'2018-09-02 04:50:24.811000']
2018-09-02 16:50:24,822;DEBUG;utils.execute;Line:91;Message:(0.001) SHOW FULL TABLES; args=None
2018-09-02 16:50:24,825;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `django_migrations` (`app`, `name`, `applied`) VALUES ('auth', '0006_require_contenttypes_0002', '2018-09-02 04:50:24.823000'); args=['auth', u'0006_require_contenttypes_0002', u'2018-09-02 04:50:24.823000']
2018-09-02 16:50:24,835;DEBUG;utils.execute;Line:91;Message:(0.001) SHOW FULL TABLES; args=None
2018-09-02 16:50:24,838;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `django_migrations` (`app`, `name`, `applied`) VALUES ('auth', '0007_alter_validators_add_error_messages', '2018-09-02 04:50:24.836000'); args=['auth', u'0007_alter_validators_add_error_messages', u'2018-09-02 04:50:24.836000']
2018-09-02 16:50:24,849;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `auth_user` MODIFY `username` varchar(150) NOT NULL; (params [])
2018-09-02 16:50:24,937;DEBUG;utils.execute;Line:91;Message:(0.087) ALTER TABLE `auth_user` MODIFY `username` varchar(150) NOT NULL; args=[]
2018-09-02 16:50:24,938;DEBUG;utils.execute;Line:91;Message:(0.001) SHOW FULL TABLES; args=None
2018-09-02 16:50:24,940;DEBUG;utils.execute;Line:91;Message:(0.000) INSERT INTO `django_migrations` (`app`, `name`, `applied`) VALUES ('auth', '0008_alter_user_username_max_length', '2018-09-02 04:50:24.939000'); args=['auth', u'0008_alter_user_username_max_length', u'2018-09-02 04:50:24.939000']
2018-09-02 16:50:24,950;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `auth_user` ADD COLUMN `cid` varchar(20) NULL; (params [])
2018-09-02 16:50:24,983;DEBUG;utils.execute;Line:91;Message:(0.033) ALTER TABLE `auth_user` ADD COLUMN `cid` varchar(20) NULL; args=[]
2018-09-02 16:50:24,986;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_user'; args=[u'auth_user']
2018-09-02 16:50:24,996;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `auth_user` ADD COLUMN `headship` varchar(20) NULL; (params [])
2018-09-02 16:50:25,045;DEBUG;utils.execute;Line:91;Message:(0.048) ALTER TABLE `auth_user` ADD COLUMN `headship` varchar(20) NULL; args=[]
2018-09-02 16:50:25,046;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_user'; args=[u'auth_user']
2018-09-02 16:50:25,048;DEBUG;utils.execute;Line:91;Message:(0.001) SHOW FULL TABLES; args=None
2018-09-02 16:50:25,049;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `django_migrations` (`app`, `name`, `applied`) VALUES ('auth', '0009_auto_20180902_1542', '2018-09-02 04:50:25.049000'); args=['auth', u'0009_auto_20180902_1542', u'2018-09-02 04:50:25.049000']
2018-09-02 16:50:25,072;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `auth_user` DROP COLUMN `cid`; (params ())
2018-09-02 16:50:25,155;DEBUG;utils.execute;Line:91;Message:(0.083) ALTER TABLE `auth_user` DROP COLUMN `cid`; args=()
2018-09-02 16:50:25,164;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `auth_user` DROP COLUMN `headship`; (params ())
2018-09-02 16:50:25,250;DEBUG;utils.execute;Line:91;Message:(0.086) ALTER TABLE `auth_user` DROP COLUMN `headship`; args=()
2018-09-02 16:50:25,257;DEBUG;schema.execute;Line:110;Message:ALTER TABLE `auth_user` ADD COLUMN `role` varchar(20) NULL; (params [])
2018-09-02 16:50:25,279;DEBUG;utils.execute;Line:91;Message:(0.021) ALTER TABLE `auth_user` ADD COLUMN `role` varchar(20) NULL; args=[]
2018-09-02 16:50:25,282;DEBUG;utils.execute;Line:91;Message:(0.003) SELECT engine FROM information_schema.tables WHERE table_name = 'auth_user'; args=[u'auth_user']
2018-09-02 16:50:25,283;DEBUG;utils.execute;Line:91;Message:(0.001) SHOW FULL TABLES; args=None
2018-09-02 16:50:25,285;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `django_migrations` (`app`, `name`, `applied`) VALUES ('auth', '0010_auto_20180902_1648', '2018-09-02 04:50:25.284000'); args=['auth', u'0010_auto_20180902_1648', u'2018-09-02 04:50:25.284000']
2018-09-02 16:50:25,289;DEBUG;schema.execute;Line:110;Message:CREATE TABLE `django_session` (`session_key` varchar(40) NOT NULL PRIMARY KEY, `session_data` longtext NOT NULL, `expire_date` datetime(6) NOT NULL); (params None)
2018-09-02 16:50:25,316;DEBUG;utils.execute;Line:91;Message:(0.028) CREATE TABLE `django_session` (`session_key` varchar(40) NOT NULL PRIMARY KEY, `session_data` longtext NOT NULL, `expire_date` datetime(6) NOT NULL); args=None
2018-09-02 16:50:25,318;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'django_session'; args=[u'django_session']
2018-09-02 16:50:25,319;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'django_session'; args=[u'django_session']
2018-09-02 16:50:25,321;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT engine FROM information_schema.tables WHERE table_name = 'django_session'; args=[u'django_session']
2018-09-02 16:50:25,321;DEBUG;schema.execute;Line:110;Message:CREATE INDEX `django_session_expire_date_a5c62663` ON `django_session` (`expire_date`); (params ())
2018-09-02 16:50:25,336;DEBUG;utils.execute;Line:91;Message:(0.015) CREATE INDEX `django_session_expire_date_a5c62663` ON `django_session` (`expire_date`); args=()
2018-09-02 16:50:25,338;DEBUG;utils.execute;Line:91;Message:(0.001) SHOW FULL TABLES; args=None
2018-09-02 16:50:25,341;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `django_migrations` (`app`, `name`, `applied`) VALUES ('sessions', '0001_initial', '2018-09-02 04:50:25.340000'); args=['sessions', u'0001_initial', u'2018-09-02 04:50:25.340000']
2018-09-02 16:50:25,342;DEBUG;utils.execute;Line:91;Message:(0.001) SHOW FULL TABLES; args=None
2018-09-02 16:50:25,344;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2018-09-02 16:50:25,352;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE (`django_content_type`.`model` = 'logentry' AND `django_content_type`.`app_label` = 'admin'); args=('logentry', 'admin')
2018-09-02 16:50:25,355;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE (`django_content_type`.`model` = 'logentry' AND `django_content_type`.`app_label` = 'admin'); args=('logentry', 'admin')
2018-09-02 16:50:25,357;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `django_content_type` (`app_label`, `model`) VALUES ('admin', 'logentry'); args=['admin', 'logentry']
2018-09-02 16:50:25,359;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `auth_permission`.`content_type_id`, `auth_permission`.`codename` FROM `auth_permission` INNER JOIN `django_content_type` ON (`auth_permission`.`content_type_id` = `django_content_type`.`id`) WHERE `auth_permission`.`content_type_id` IN (1) ORDER BY `django_content_type`.`app_label` ASC, `django_content_type`.`model` ASC, `auth_permission`.`codename` ASC; args=(1,)
2018-09-02 16:50:25,362;DEBUG;utils.execute;Line:91;Message:(0.002) INSERT INTO `auth_permission` (`name`, `content_type_id`, `codename`) VALUES ('Can add log entry', 1, 'add_logentry'), ('Can change log entry', 1, 'change_logentry'), ('Can delete log entry', 1, 'delete_logentry'); args=(u'Can add log entry', 1, u'add_logentry', u'Can change log entry', 1, u'change_logentry', u'Can delete log entry', 1, u'delete_logentry')
2018-09-02 16:50:25,365;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE `django_content_type`.`app_label` = 'admin'; args=('admin',)
2018-09-02 16:50:25,367;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE (`django_content_type`.`model` = 'group' AND `django_content_type`.`app_label` = 'auth'); args=('group', 'auth')
2018-09-02 16:50:25,368;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE (`django_content_type`.`model` = 'group' AND `django_content_type`.`app_label` = 'auth'); args=('group', 'auth')
2018-09-02 16:50:25,371;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `django_content_type` (`app_label`, `model`) VALUES ('auth', 'group'); args=['auth', 'group']
2018-09-02 16:50:25,372;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE (`django_content_type`.`model` = 'permission' AND `django_content_type`.`app_label` = 'auth'); args=('permission', 'auth')
2018-09-02 16:50:25,374;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE (`django_content_type`.`model` = 'permission' AND `django_content_type`.`app_label` = 'auth'); args=('permission', 'auth')
2018-09-02 16:50:25,377;DEBUG;utils.execute;Line:91;Message:(0.002) INSERT INTO `django_content_type` (`app_label`, `model`) VALUES ('auth', 'permission'); args=['auth', 'permission']
2018-09-02 16:50:25,378;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE (`django_content_type`.`model` = 'user' AND `django_content_type`.`app_label` = 'auth'); args=('user', 'auth')
2018-09-02 16:50:25,381;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE (`django_content_type`.`model` = 'user' AND `django_content_type`.`app_label` = 'auth'); args=('user', 'auth')
2018-09-02 16:50:25,382;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `django_content_type` (`app_label`, `model`) VALUES ('auth', 'user'); args=['auth', 'user']
2018-09-02 16:50:25,385;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_permission`.`content_type_id`, `auth_permission`.`codename` FROM `auth_permission` INNER JOIN `django_content_type` ON (`auth_permission`.`content_type_id` = `django_content_type`.`id`) WHERE `auth_permission`.`content_type_id` IN (2, 3, 4) ORDER BY `django_content_type`.`app_label` ASC, `django_content_type`.`model` ASC, `auth_permission`.`codename` ASC; args=(2, 3, 4)
2018-09-02 16:50:25,388;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `auth_permission` (`name`, `content_type_id`, `codename`) VALUES ('Can add group', 2, 'add_group'), ('Can change group', 2, 'change_group'), ('Can delete group', 2, 'delete_group'), ('Can add permission', 3, 'add_permission'), ('Can change permission', 3, 'change_permission'), ('Can delete permission', 3, 'delete_permission'), ('Can add user', 4, 'add_user'), ('Can change user', 4, 'change_user'), ('Can delete user', 4, 'delete_user'); args=(u'Can add group', 2, u'add_group', u'Can change group', 2, u'change_group', u'Can delete group', 2, u'delete_group', u'Can add permission', 3, u'add_permission', u'Can change permission', 3, u'change_permission', u'Can delete permission', 3, u'delete_permission', u'Can add user', 4, u'add_user', u'Can change user', 4, u'change_user', u'Can delete user', 4, u'delete_user')
2018-09-02 16:50:25,391;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE `django_content_type`.`app_label` = 'auth'; args=('auth',)
2018-09-02 16:50:25,392;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE (`django_content_type`.`model` = 'contenttype' AND `django_content_type`.`app_label` = 'contenttypes'); args=('contenttype', 'contenttypes')
2018-09-02 16:50:25,394;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE (`django_content_type`.`model` = 'contenttype' AND `django_content_type`.`app_label` = 'contenttypes'); args=('contenttype', 'contenttypes')
2018-09-02 16:50:25,394;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `django_content_type` (`app_label`, `model`) VALUES ('contenttypes', 'contenttype'); args=['contenttypes', 'contenttype']
2018-09-02 16:50:25,398;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_permission`.`content_type_id`, `auth_permission`.`codename` FROM `auth_permission` INNER JOIN `django_content_type` ON (`auth_permission`.`content_type_id` = `django_content_type`.`id`) WHERE `auth_permission`.`content_type_id` IN (5) ORDER BY `django_content_type`.`app_label` ASC, `django_content_type`.`model` ASC, `auth_permission`.`codename` ASC; args=(5,)
2018-09-02 16:50:25,401;DEBUG;utils.execute;Line:91;Message:(0.000) INSERT INTO `auth_permission` (`name`, `content_type_id`, `codename`) VALUES ('Can add content type', 5, 'add_contenttype'), ('Can change content type', 5, 'change_contenttype'), ('Can delete content type', 5, 'delete_contenttype'); args=(u'Can add content type', 5, u'add_contenttype', u'Can change content type', 5, u'change_contenttype', u'Can delete content type', 5, u'delete_contenttype')
2018-09-02 16:50:25,403;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE `django_content_type`.`app_label` = 'contenttypes'; args=('contenttypes',)
2018-09-02 16:50:25,404;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE (`django_content_type`.`model` = 'session' AND `django_content_type`.`app_label` = 'sessions'); args=('session', 'sessions')
2018-09-02 16:50:25,405;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE (`django_content_type`.`model` = 'session' AND `django_content_type`.`app_label` = 'sessions'); args=('session', 'sessions')
2018-09-02 16:50:25,407;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `django_content_type` (`app_label`, `model`) VALUES ('sessions', 'session'); args=['sessions', 'session']
2018-09-02 16:50:25,410;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `auth_permission`.`content_type_id`, `auth_permission`.`codename` FROM `auth_permission` INNER JOIN `django_content_type` ON (`auth_permission`.`content_type_id` = `django_content_type`.`id`) WHERE `auth_permission`.`content_type_id` IN (6) ORDER BY `django_content_type`.`app_label` ASC, `django_content_type`.`model` ASC, `auth_permission`.`codename` ASC; args=(6,)
2018-09-02 16:50:25,411;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `auth_permission` (`name`, `content_type_id`, `codename`) VALUES ('Can add session', 6, 'add_session'), ('Can change session', 6, 'change_session'), ('Can delete session', 6, 'delete_session'); args=(u'Can add session', 6, u'add_session', u'Can change session', 6, u'change_session', u'Can delete session', 6, u'delete_session')
2018-09-02 16:50:25,414;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE `django_content_type`.`app_label` = 'sessions'; args=('sessions',)
2018-09-02 16:58:58,025;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 16:58:58,026;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT VERSION(); args=None
2018-09-02 16:58:58,068;DEBUG;utils.execute;Line:91;Message:(0.001) SHOW FULL TABLES; args=None
2018-09-02 16:58:58,072;DEBUG;utils.execute;Line:91;Message:(0.002) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2018-09-02 16:59:09,733;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 16:59:09,733;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 16:59:09,739;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 've3zjhvqaotgtavzymd44je7mu7cgsd9' AND `django_session`.`expire_date` > '2018-09-02 04:59:09.730000'); args=('ve3zjhvqaotgtavzymd44je7mu7cgsd9', u'2018-09-02 04:59:09.730000')
2018-09-02 16:59:09,740;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT (1) AS `a` FROM `django_session` WHERE `django_session`.`session_key` = '8a26w5d7tzwd0cwhpds5tlof2tqgiuoe' LIMIT 1; args=(u'8a26w5d7tzwd0cwhpds5tlof2tqgiuoe',)
2018-09-02 16:59:09,743;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `django_session` (`session_key`, `session_data`, `expire_date`) VALUES ('8a26w5d7tzwd0cwhpds5tlof2tqgiuoe', 'MmNmNTU3YTQxY2M1MjFhMzFiMzI5MzhmZjJiZTY3ZmE2NTU3ZmVkODp7fQ==', '2018-09-16 04:59:09.742000'); args=(u'8a26w5d7tzwd0cwhpds5tlof2tqgiuoe', u'MmNmNTU3YTQxY2M1MjFhMzFiMzI5MzhmZjJiZTY3ZmE2NTU3ZmVkODp7fQ==', u'2018-09-16 04:59:09.742000')
2018-09-02 16:59:09,829;INFO;basehttp.log_message;Line:124;Message:"GET / HTTP/1.1" 200 1716
2018-09-02 16:59:24,506;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 16:59:24,507;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 16:59:24,509;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = '8a26w5d7tzwd0cwhpds5tlof2tqgiuoe' AND `django_session`.`expire_date` > '2018-09-02 04:59:24.503000'); args=('8a26w5d7tzwd0cwhpds5tlof2tqgiuoe', u'2018-09-02 04:59:24.503000')
2018-09-02 16:59:24,520;DEBUG;utils.execute;Line:91;Message:(0.002) UPDATE `django_session` SET `session_data` = 'MmNmNTU3YTQxY2M1MjFhMzFiMzI5MzhmZjJiZTY3ZmE2NTU3ZmVkODp7fQ==', `expire_date` = '2018-09-16 04:59:24.519000' WHERE `django_session`.`session_key` = '8a26w5d7tzwd0cwhpds5tlof2tqgiuoe'; args=(u'MmNmNTU3YTQxY2M1MjFhMzFiMzI5MzhmZjJiZTY3ZmE2NTU3ZmVkODp7fQ==', u'2018-09-16 04:59:24.519000', '8a26w5d7tzwd0cwhpds5tlof2tqgiuoe')
2018-09-02 16:59:24,619;INFO;basehttp.log_message;Line:124;Message:"GET /admin/ HTTP/1.1" 302 0
2018-09-02 16:59:24,625;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 16:59:24,625;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 16:59:24,627;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = '8a26w5d7tzwd0cwhpds5tlof2tqgiuoe' AND `django_session`.`expire_date` > '2018-09-02 04:59:24.623000'); args=('8a26w5d7tzwd0cwhpds5tlof2tqgiuoe', u'2018-09-02 04:59:24.623000')
2018-09-02 16:59:24,638;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'is_popup' in template 'admin/login.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [is_popup] in u"[{'False': False, 'None': None, 'True': True}, {u'csrf_token': <SimpleLazyObject: <function _get_val at 0x0000000004F7A5F8>>, 'user': <SimpleLazyObject: <django.contrib.auth.models.AnonymousUser object at 0x0000000004F28F28>>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x0000000004F78400>, 'DEFAULT_MESSAGE_LEVELS': {'DEBUG': 10, 'INFO': 20, 'WARNING': 30, 'SUCCESS': 25, 'ERROR': 40}, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x0000000004F28B70>, u'request': <WSGIRequest: GET '/admin/login/?next=/admin/'>}, {}, {'username': u'', 'app_path': u'/admin/login/?next=/admin/', 'available_apps': [], 'site_name': '127.0.0.1:8000', 'form': <AdminAuthenticationForm bound=False, valid=Unknown, fields=(username;password)>, 'title': u'Log in', 'site_header': u'Django administration', 'site': <django.contrib.sites.requests.RequestSite object at 0x0000000004F43128>, 'next': u'/admin/', 'site_title': u'Django site admin', u'LANGUAGE_CODE': 'en-us', 'has_permission': False, 'site_url': '/', u'LANGUAGE_BIDI': False, u'view': <django.contrib.auth.views.LoginView object at 0x0000000004F289E8>}]"
2018-09-02 16:59:24,664;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'is_popup' in template 'admin/login.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [is_popup] in u"[{'False': False, 'None': None, 'True': True}, {u'csrf_token': <SimpleLazyObject: <function _get_val at 0x0000000004F7A5F8>>, 'user': <SimpleLazyObject: <django.contrib.auth.models.AnonymousUser object at 0x0000000004F28F28>>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x0000000004F78400>, 'DEFAULT_MESSAGE_LEVELS': {'DEBUG': 10, 'INFO': 20, 'WARNING': 30, 'SUCCESS': 25, 'ERROR': 40}, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x0000000004F28B70>, u'request': <WSGIRequest: GET '/admin/login/?next=/admin/'>}, {}, {'username': u'', 'app_path': u'/admin/login/?next=/admin/', 'available_apps': [], 'site_name': '127.0.0.1:8000', 'form': <AdminAuthenticationForm bound=False, valid=Unknown, fields=(username;password)>, 'title': u'Log in', 'site_header': u'Django administration', 'site': <django.contrib.sites.requests.RequestSite object at 0x0000000004F43128>, 'next': u'/admin/', 'site_title': u'Django site admin', u'LANGUAGE_CODE': 'en-us', 'has_permission': False, 'site_url': '/', u'LANGUAGE_BIDI': False, u'view': <django.contrib.auth.views.LoginView object at 0x0000000004F289E8>}]"
2018-09-02 16:59:24,673;DEBUG;utils.execute;Line:91;Message:(0.000) UPDATE `django_session` SET `session_data` = 'MmNmNTU3YTQxY2M1MjFhMzFiMzI5MzhmZjJiZTY3ZmE2NTU3ZmVkODp7fQ==', `expire_date` = '2018-09-16 04:59:24.672000' WHERE `django_session`.`session_key` = '8a26w5d7tzwd0cwhpds5tlof2tqgiuoe'; args=(u'MmNmNTU3YTQxY2M1MjFhMzFiMzI5MzhmZjJiZTY3ZmE2NTU3ZmVkODp7fQ==', u'2018-09-16 04:59:24.672000', '8a26w5d7tzwd0cwhpds5tlof2tqgiuoe')
2018-09-02 16:59:24,676;INFO;basehttp.log_message;Line:124;Message:"GET /admin/login/?next=/admin/ HTTP/1.1" 200 1650
2018-09-02 17:00:02,079;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:00:02,081;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:00:02,108;DEBUG;utils.execute;Line:91;Message:(0.002) SHOW FULL TABLES; args=None
2018-09-02 17:00:02,109;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2018-09-02 17:00:02,112;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`username` = 'xingtong'; args=(u'xingtong',)
2018-09-02 17:00:18,661;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`username` = 'admin'; args=('admin',)
2018-09-02 17:01:05,892;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `auth_user` (`password`, `last_login`, `is_superuser`, `username`, `first_name`, `last_name`, `email`, `is_staff`, `is_active`, `date_joined`, `role`) VALUES ('pbkdf2_sha256$36000$jRte1HzyiLxd$kg372b+L/Z59v2ZgeUJxEqvsA0paL5itYZgkr5BqsOc=', NULL, 1, 'admin', '', '', 'x1980p@gmail.com', 1, 1, '2018-09-02 05:01:05.860000', NULL); args=[u'pbkdf2_sha256$36000$jRte1HzyiLxd$kg372b+L/Z59v2ZgeUJxEqvsA0paL5itYZgkr5BqsOc=', None, True, u'admin', u'', u'', u'x1980p@gmail.com', True, True, u'2018-09-02 05:01:05.860000', None]
2018-09-02 17:01:17,342;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:01:17,344;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT VERSION(); args=None
2018-09-02 17:01:17,346;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = '8a26w5d7tzwd0cwhpds5tlof2tqgiuoe' AND `django_session`.`expire_date` > '2018-09-02 05:01:17.339000'); args=('8a26w5d7tzwd0cwhpds5tlof2tqgiuoe', u'2018-09-02 05:01:17.339000')
2018-09-02 17:01:17,351;DEBUG;utils.execute;Line:91;Message:(0.002) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`username` = 'admin'; args=(u'admin',)
2018-09-02 17:01:17,384;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT (1) AS `a` FROM `django_session` WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' LIMIT 1; args=(u'li4w1895i6ez8vw5iyzbs07k57o0yv8q',)
2018-09-02 17:01:17,385;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `django_session` (`session_key`, `session_data`, `expire_date`) VALUES ('li4w1895i6ez8vw5iyzbs07k57o0yv8q', 'MmNmNTU3YTQxY2M1MjFhMzFiMzI5MzhmZjJiZTY3ZmE2NTU3ZmVkODp7fQ==', '2018-09-16 05:01:17.385000'); args=(u'li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'MmNmNTU3YTQxY2M1MjFhMzFiMzI5MzhmZjJiZTY3ZmE2NTU3ZmVkODp7fQ==', u'2018-09-16 05:01:17.385000')
2018-09-02 17:01:17,388;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE `django_session`.`session_key` = '8a26w5d7tzwd0cwhpds5tlof2tqgiuoe'; args=('8a26w5d7tzwd0cwhpds5tlof2tqgiuoe',)
2018-09-02 17:01:17,390;DEBUG;utils.execute;Line:91;Message:(0.001) DELETE FROM `django_session` WHERE `django_session`.`session_key` IN ('8a26w5d7tzwd0cwhpds5tlof2tqgiuoe'); args=(u'8a26w5d7tzwd0cwhpds5tlof2tqgiuoe',)
2018-09-02 17:01:17,395;DEBUG;utils.execute;Line:91;Message:(0.000) UPDATE `auth_user` SET `last_login` = '2018-09-02 05:01:17.395000' WHERE `auth_user`.`id` = 1; args=(u'2018-09-02 05:01:17.395000', 1)
2018-09-02 17:01:17,401;DEBUG;utils.execute;Line:91;Message:(0.001) UPDATE `django_session` SET `session_data` = 'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', `expire_date` = '2018-09-16 05:01:17.400000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', u'2018-09-16 05:01:17.400000', u'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:01:17,403;INFO;basehttp.log_message;Line:124;Message:"POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
2018-09-02 17:01:17,407;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:01:17,407;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:01:17,410;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:01:17.406000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:01:17.406000')
2018-09-02 17:01:17,411;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:01:17,420;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'is_popup' in template 'admin/index.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [is_popup] in u"[{'False': False, 'None': None, 'True': True}, {u'csrf_token': <SimpleLazyObject: <function _get_val at 0x0000000004F7A668>>, 'user': <SimpleLazyObject: <User: admin>>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x0000000004F84550>, 'DEFAULT_MESSAGE_LEVELS': {'DEBUG': 10, 'INFO': 20, 'WARNING': 30, 'SUCCESS': 25, 'ERROR': 40}, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x0000000004F67780>, u'request': <WSGIRequest: GET '/admin/'>}, {}, {'site_header': u'Django administration', 'available_apps': [{'app_url': u'/admin/auth/', 'models': [{'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/group/', 'object_name': 'Group', 'name': u'Groups', 'add_url': u'/admin/auth/group/add/'}, {'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/user/', 'object_name': 'User', 'name': u'Users', 'add_url': u'/admin/auth/user/add/'}], 'has_module_perms': True, 'name': u'Authentication and Authorization', 'app_label': 'auth'}], 'site_title': u'Django site admin', 'title': u'Site administration', 'has_permission': True, 'app_list': [{'app_url': u'/admin/auth/', 'models': [{'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/group/', 'object_name': 'Group', 'name': u'Groups', 'add_url': u'/admin/auth/group/add/'}, {'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/user/', 'object_name': 'User', 'name': u'Users', 'add_url': u'/admin/auth/user/add/'}], 'has_module_perms': True, 'name': u'Authentication and Authorization', 'app_label': 'auth'}], u'LANGUAGE_CODE': 'en-us', u'LANGUAGE_BIDI': False, 'site_url': '/'}]"
2018-09-02 17:01:17,421;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'is_popup' in template 'admin/index.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [is_popup] in u"[{'False': False, 'None': None, 'True': True}, {u'csrf_token': <SimpleLazyObject: <function _get_val at 0x0000000004F7A668>>, 'user': <SimpleLazyObject: <User: admin>>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x0000000004F84550>, 'DEFAULT_MESSAGE_LEVELS': {'DEBUG': 10, 'INFO': 20, 'WARNING': 30, 'SUCCESS': 25, 'ERROR': 40}, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x0000000004F67780>, u'request': <WSGIRequest: GET '/admin/'>}, {}, {'site_header': u'Django administration', 'available_apps': [{'app_url': u'/admin/auth/', 'models': [{'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/group/', 'object_name': 'Group', 'name': u'Groups', 'add_url': u'/admin/auth/group/add/'}, {'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/user/', 'object_name': 'User', 'name': u'Users', 'add_url': u'/admin/auth/user/add/'}], 'has_module_perms': True, 'name': u'Authentication and Authorization', 'app_label': 'auth'}], 'site_title': u'Django site admin', 'title': u'Site administration', 'has_permission': True, 'app_list': [{'app_url': u'/admin/auth/', 'models': [{'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/group/', 'object_name': 'Group', 'name': u'Groups', 'add_url': u'/admin/auth/group/add/'}, {'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/user/', 'object_name': 'User', 'name': u'Users', 'add_url': u'/admin/auth/user/add/'}], 'has_module_perms': True, 'name': u'Authentication and Authorization', 'app_label': 'auth'}], u'LANGUAGE_CODE': 'en-us', u'LANGUAGE_BIDI': False, 'site_url': '/'}]"
2018-09-02 17:01:17,427;DEBUG;utils.execute;Line:91;Message:(0.002) SELECT `django_admin_log`.`id`, `django_admin_log`.`action_time`, `django_admin_log`.`user_id`, `django_admin_log`.`content_type_id`, `django_admin_log`.`object_id`, `django_admin_log`.`object_repr`, `django_admin_log`.`action_flag`, `django_admin_log`.`change_message`, `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role`, `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_admin_log` INNER JOIN `auth_user` ON (`django_admin_log`.`user_id` = `auth_user`.`id`) LEFT OUTER JOIN `django_content_type` ON (`django_admin_log`.`content_type_id` = `django_content_type`.`id`) WHERE `django_admin_log`.`user_id` = 1 ORDER BY `django_admin_log`.`action_time` DESC LIMIT 10; args=(1,)
2018-09-02 17:01:17,430;DEBUG;utils.execute;Line:91;Message:(0.000) UPDATE `django_session` SET `session_data` = 'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', `expire_date` = '2018-09-16 05:01:17.430000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', u'2018-09-16 05:01:17.430000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:01:17,434;INFO;basehttp.log_message;Line:124;Message:"GET /admin/ HTTP/1.1" 200 2779
2018-09-02 17:01:20,772;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:01:20,773;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT VERSION(); args=None
2018-09-02 17:01:20,776;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:01:20.769000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:01:20.769000')
2018-09-02 17:01:20,778;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:01:20,780;DEBUG;utils.execute;Line:91;Message:(0.002) SELECT `auth_group`.`id`, `auth_group`.`name` FROM `auth_group`; args=()
2018-09-02 17:01:20,782;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT COUNT(*) AS `__count` FROM `auth_user`; args=()
2018-09-02 17:01:20,783;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT COUNT(*) AS `__count` FROM `auth_user`; args=()
2018-09-02 17:01:20,786;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` ORDER BY `auth_user`.`username` ASC, `auth_user`.`id` DESC; args=()
2018-09-02 17:01:20,802;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'errors' in template 'admin/change_list.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [errors] in u'None'
2018-09-02 17:01:20,805;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'show' in template 'admin/change_list.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [show] in u"[{'False': False, 'None': None, 'True': True, 'csrf_token': <SimpleLazyObject: <function _get_val at 0x0000000004F35518>>}]"
2018-09-02 17:01:20,809;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'is_multipart' in template 'admin/change_list.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [is_multipart] in u'None'
2018-09-02 17:01:20,823;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'non_field_errors' in template 'admin/change_list.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [non_field_errors] in u'None'
2018-09-02 17:01:20,828;DEBUG;utils.execute;Line:91;Message:(0.001) UPDATE `django_session` SET `session_data` = 'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', `expire_date` = '2018-09-16 05:01:20.827000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', u'2018-09-16 05:01:20.827000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:01:20,849;INFO;basehttp.log_message;Line:124;Message:"GET /admin/auth/user/ HTTP/1.1" 200 6459
2018-09-02 17:01:20,882;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:01:20,882;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:01:20,894;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:01:20.877000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:01:20.877000')
2018-09-02 17:01:20,895;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:01:20,900;DEBUG;utils.execute;Line:91;Message:(0.001) UPDATE `django_session` SET `session_data` = 'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', `expire_date` = '2018-09-16 05:01:20.898000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', u'2018-09-16 05:01:20.898000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:01:20,901;INFO;basehttp.log_message;Line:124;Message:"GET /admin/jsi18n/ HTTP/1.1" 200 3217
2018-09-02 17:01:21,457;INFO;basehttp.log_message;Line:124;Message:"GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 258648
2018-09-02 17:01:21,471;INFO;basehttp.log_message;Line:124;Message:"GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1538
2018-09-02 17:01:21,473;INFO;basehttp.log_message;Line:124;Message:"GET /static/admin/js/actions.js HTTP/1.1" 200 6501
2018-09-02 17:01:21,476;INFO;basehttp.log_message;Line:124;Message:"GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436
2018-09-02 17:01:21,480;INFO;basehttp.log_message;Line:124;Message:"GET /static/admin/img/search.svg HTTP/1.1" 200 458
2018-09-02 17:01:21,492;INFO;basehttp.log_message;Line:124;Message:"GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 128820
2018-09-02 17:01:27,779;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:01:27,780;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:01:27,783;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:01:27.777000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:01:27.777000')
2018-09-02 17:01:27,786;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:01:27,798;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'is_popup' in template 'admin/app_index.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [is_popup] in u"[{'False': False, 'None': None, 'True': True}, {u'csrf_token': <SimpleLazyObject: <function _get_val at 0x0000000004FEC668>>, 'user': <SimpleLazyObject: <User: admin>>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x0000000004FF8908>, 'DEFAULT_MESSAGE_LEVELS': {'DEBUG': 10, 'INFO': 20, 'WARNING': 30, 'SUCCESS': 25, 'ERROR': 40}, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x0000000005067A90>, u'request': <WSGIRequest: GET '/admin/auth/'>}, {}, {'available_apps': [{'app_url': u'/admin/auth/', 'models': [{'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/group/', 'object_name': 'Group', 'name': u'Groups', 'add_url': u'/admin/auth/group/add/'}, {'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/user/', 'object_name': 'User', 'name': u'Users', 'add_url': u'/admin/auth/user/add/'}], 'has_module_perms': True, 'name': u'Authentication and Authorization', 'app_label': 'auth'}], 'app_list': [{'app_url': u'/admin/auth/', 'models': [{'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/group/', 'object_name': 'Group', 'name': u'Groups', 'add_url': u'/admin/auth/group/add/'}, {'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/user/', 'object_name': 'User', 'name': u'Users', 'add_url': u'/admin/auth/user/add/'}], 'has_module_perms': True, 'name': u'Authentication and Authorization', 'app_label': 'auth'}], 'title': u'Authentication and Authorization administration', 'site_header': u'Django administration', u'LANGUAGE_CODE': 'en-us', 'site_url': '/', 'site_title': u'Django site admin', 'app_label': u'auth', 'has_permission': True, u'LANGUAGE_BIDI': False}]"
2018-09-02 17:01:27,799;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'is_popup' in template 'admin/app_index.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [is_popup] in u"[{'False': False, 'None': None, 'True': True}, {u'csrf_token': <SimpleLazyObject: <function _get_val at 0x0000000004FEC668>>, 'user': <SimpleLazyObject: <User: admin>>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x0000000004FF8908>, 'DEFAULT_MESSAGE_LEVELS': {'DEBUG': 10, 'INFO': 20, 'WARNING': 30, 'SUCCESS': 25, 'ERROR': 40}, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x0000000005067A90>, u'request': <WSGIRequest: GET '/admin/auth/'>}, {}, {'available_apps': [{'app_url': u'/admin/auth/', 'models': [{'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/group/', 'object_name': 'Group', 'name': u'Groups', 'add_url': u'/admin/auth/group/add/'}, {'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/user/', 'object_name': 'User', 'name': u'Users', 'add_url': u'/admin/auth/user/add/'}], 'has_module_perms': True, 'name': u'Authentication and Authorization', 'app_label': 'auth'}], 'app_list': [{'app_url': u'/admin/auth/', 'models': [{'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/group/', 'object_name': 'Group', 'name': u'Groups', 'add_url': u'/admin/auth/group/add/'}, {'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/user/', 'object_name': 'User', 'name': u'Users', 'add_url': u'/admin/auth/user/add/'}], 'has_module_perms': True, 'name': u'Authentication and Authorization', 'app_label': 'auth'}], 'title': u'Authentication and Authorization administration', 'site_header': u'Django administration', u'LANGUAGE_CODE': 'en-us', 'site_url': '/', 'site_title': u'Django site admin', 'app_label': u'auth', 'has_permission': True, u'LANGUAGE_BIDI': False}]"
2018-09-02 17:01:27,803;DEBUG;utils.execute;Line:91;Message:(0.000) UPDATE `django_session` SET `session_data` = 'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', `expire_date` = '2018-09-16 05:01:27.802000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', u'2018-09-16 05:01:27.802000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:01:27,808;INFO;basehttp.log_message;Line:124;Message:"GET /admin/auth/ HTTP/1.1" 200 2705
2018-09-02 17:01:28,842;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:01:28,842;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:01:28,845;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:01:28.839000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:01:28.839000')
2018-09-02 17:01:28,848;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:01:28,848;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT COUNT(*) AS `__count` FROM `auth_group`; args=()
2018-09-02 17:01:28,849;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT COUNT(*) AS `__count` FROM `auth_group`; args=()
2018-09-02 17:01:28,851;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_group`.`id`, `auth_group`.`name` FROM `auth_group` ORDER BY `auth_group`.`name` ASC, `auth_group`.`id` DESC; args=()
2018-09-02 17:01:28,877;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'errors' in template 'admin/change_list.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [errors] in u'None'
2018-09-02 17:01:28,880;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'show' in template 'admin/change_list.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [show] in u"[{'False': False, 'None': None, 'True': True, 'csrf_token': <SimpleLazyObject: <function _get_val at 0x0000000004F7A668>>}]"
2018-09-02 17:01:28,880;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'is_multipart' in template 'admin/change_list.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [is_multipart] in u'None'
2018-09-02 17:01:28,885;DEBUG;utils.execute;Line:91;Message:(0.000) UPDATE `django_session` SET `session_data` = 'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', `expire_date` = '2018-09-16 05:01:28.885000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', u'2018-09-16 05:01:28.885000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:01:28,888;INFO;basehttp.log_message;Line:124;Message:"GET /admin/auth/group/ HTTP/1.1" 200 3379
2018-09-02 17:01:28,907;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:01:28,907;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:01:28,913;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:01:28.904000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:01:28.904000')
2018-09-02 17:01:28,914;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:01:28,917;DEBUG;utils.execute;Line:91;Message:(0.000) UPDATE `django_session` SET `session_data` = 'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', `expire_date` = '2018-09-16 05:01:28.916000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', u'2018-09-16 05:01:28.916000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:01:28,918;INFO;basehttp.log_message;Line:124;Message:"GET /admin/jsi18n/ HTTP/1.1" 200 3217
2018-09-02 17:01:31,515;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:01:31,516;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:01:31,519;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:01:31.512000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:01:31.512000')
2018-09-02 17:01:31,520;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:01:31,526;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE (`django_content_type`.`model` = 'group' AND `django_content_type`.`app_label` = 'auth'); args=('group', 'auth')
2018-09-02 17:01:31,546;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_permission`.`id`, `auth_permission`.`name`, `auth_permission`.`content_type_id`, `auth_permission`.`codename`, `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `auth_permission` INNER JOIN `django_content_type` ON (`auth_permission`.`content_type_id` = `django_content_type`.`id`) ORDER BY `django_content_type`.`app_label` ASC, `django_content_type`.`model` ASC, `auth_permission`.`codename` ASC; args=()
2018-09-02 17:01:31,552;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'can_change_related' in template 'admin/widgets/related_widget_wrapper.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [can_change_related] in u'[{\'False\': False, \'None\': None, \'True\': True}, {u\'model\': u\'permission\', u\'url_params\': u\'_to_field=id&_popup=1\', u\'rendered_widget\': u\'<select name="permissions" data-field-name="permissions" multiple="multiple" id="id_permissions" data-is-stacked="0" class="selectfilter">\\n  <option value="1">admin | log entry | Can add log entry</option>\\n\\n  <option value="2">admin | log entry | Can change log entry</option>\\n\\n  <option value="3">admin | log entry | Can delete log entry</option>\\n\\n  <option value="4">auth | group | Can add group</option>\\n\\n  <option value="5">auth | group | Can change group</option>\\n\\n  <option value="6">auth | group | Can delete group</option>\\n\\n  <option value="7">auth | permission | Can add permission</option>\\n\\n  <option value="8">auth | permission | Can change permission</option>\\n\\n  <option value="9">auth | permission | Can delete permission</option>\\n\\n  <option value="10">auth | user | Can add user</option>\\n\\n  <option value="11">auth | user | Can change user</option>\\n\\n  <option value="12">auth | user | Can delete user</option>\\n\\n  <option value="13">contenttypes | content type | Can add content type</option>\\n\\n  <option value="14">contenttypes | content type | Can change content type</option>\\n\\n  <option value="15">contenttypes | content type | Can delete content type</option>\\n\\n  <option value="16">sessions | session | Can add session</option>\\n\\n  <option value="17">sessions | session | Can change session</option>\\n\\n  <option value="18">sessions | session | Can delete session</option>\\n\\n</select>\', u\'name\': \'permissions\'}, {\'block\': <Block Node: links. Contents: [<TextNode: u\'\\n        \'>, <django.template.defaulttags.SpacelessNode object at 0x00000000050DEE48>, <TextNode: u\'\\n    \'>]>}]'
2018-09-02 17:01:31,552;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'can_add_related' in template 'admin/widgets/related_widget_wrapper.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [can_add_related] in u'[{\'False\': False, \'None\': None, \'True\': True}, {u\'model\': u\'permission\', u\'url_params\': u\'_to_field=id&_popup=1\', u\'rendered_widget\': u\'<select name="permissions" data-field-name="permissions" multiple="multiple" id="id_permissions" data-is-stacked="0" class="selectfilter">\\n  <option value="1">admin | log entry | Can add log entry</option>\\n\\n  <option value="2">admin | log entry | Can change log entry</option>\\n\\n  <option value="3">admin | log entry | Can delete log entry</option>\\n\\n  <option value="4">auth | group | Can add group</option>\\n\\n  <option value="5">auth | group | Can change group</option>\\n\\n  <option value="6">auth | group | Can delete group</option>\\n\\n  <option value="7">auth | permission | Can add permission</option>\\n\\n  <option value="8">auth | permission | Can change permission</option>\\n\\n  <option value="9">auth | permission | Can delete permission</option>\\n\\n  <option value="10">auth | user | Can add user</option>\\n\\n  <option value="11">auth | user | Can change user</option>\\n\\n  <option value="12">auth | user | Can delete user</option>\\n\\n  <option value="13">contenttypes | content type | Can add content type</option>\\n\\n  <option value="14">contenttypes | content type | Can change content type</option>\\n\\n  <option value="15">contenttypes | content type | Can delete content type</option>\\n\\n  <option value="16">sessions | session | Can add session</option>\\n\\n  <option value="17">sessions | session | Can change session</option>\\n\\n  <option value="18">sessions | session | Can delete session</option>\\n\\n</select>\', u\'name\': \'permissions\'}, {\'block\': <Block Node: links. Contents: [<TextNode: u\'\\n        \'>, <django.template.defaulttags.SpacelessNode object at 0x00000000050DEE48>, <TextNode: u\'\\n    \'>]>}]'
2018-09-02 17:01:31,553;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'can_delete_related' in template 'admin/widgets/related_widget_wrapper.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [can_delete_related] in u'[{\'False\': False, \'None\': None, \'True\': True}, {u\'model\': u\'permission\', u\'url_params\': u\'_to_field=id&_popup=1\', u\'rendered_widget\': u\'<select name="permissions" data-field-name="permissions" multiple="multiple" id="id_permissions" data-is-stacked="0" class="selectfilter">\\n  <option value="1">admin | log entry | Can add log entry</option>\\n\\n  <option value="2">admin | log entry | Can change log entry</option>\\n\\n  <option value="3">admin | log entry | Can delete log entry</option>\\n\\n  <option value="4">auth | group | Can add group</option>\\n\\n  <option value="5">auth | group | Can change group</option>\\n\\n  <option value="6">auth | group | Can delete group</option>\\n\\n  <option value="7">auth | permission | Can add permission</option>\\n\\n  <option value="8">auth | permission | Can change permission</option>\\n\\n  <option value="9">auth | permission | Can delete permission</option>\\n\\n  <option value="10">auth | user | Can add user</option>\\n\\n  <option value="11">auth | user | Can change user</option>\\n\\n  <option value="12">auth | user | Can delete user</option>\\n\\n  <option value="13">contenttypes | content type | Can add content type</option>\\n\\n  <option value="14">contenttypes | content type | Can change content type</option>\\n\\n  <option value="15">contenttypes | content type | Can delete content type</option>\\n\\n  <option value="16">sessions | session | Can add session</option>\\n\\n  <option value="17">sessions | session | Can change session</option>\\n\\n  <option value="18">sessions | session | Can delete session</option>\\n\\n</select>\', u\'name\': \'permissions\'}, {\'block\': <Block Node: links. Contents: [<TextNode: u\'\\n        \'>, <django.template.defaulttags.SpacelessNode object at 0x00000000050DEE48>, <TextNode: u\'\\n    \'>]>}]'
2018-09-02 17:01:31,559;DEBUG;utils.execute;Line:91;Message:(0.001) UPDATE `django_session` SET `session_data` = 'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', `expire_date` = '2018-09-16 05:01:31.558000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', u'2018-09-16 05:01:31.558000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:01:31,573;INFO;basehttp.log_message;Line:124;Message:"GET /admin/auth/group/add/ HTTP/1.1" 200 5973
2018-09-02 17:01:31,605;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:01:31,605;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:01:31,608;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:01:31.602000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:01:31.602000')
2018-09-02 17:01:31,609;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:01:31,612;DEBUG;utils.execute;Line:91;Message:(0.001) UPDATE `django_session` SET `session_data` = 'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', `expire_date` = '2018-09-16 05:01:31.612000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', u'2018-09-16 05:01:31.612000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:01:31,618;INFO;basehttp.log_message;Line:124;Message:"GET /admin/jsi18n/ HTTP/1.1" 200 3217
2018-09-02 17:01:31,664;INFO;basehttp.log_message;Line:124;Message:"GET /static/admin/js/change_form.js HTTP/1.1" 200 712
2018-09-02 17:01:31,707;INFO;basehttp.log_message;Line:124;Message:"GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 495
2018-09-02 17:02:22,151;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:02:22,151;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT VERSION(); args=None
2018-09-02 17:02:22,154;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:02:22.148000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:02:22.148000')
2018-09-02 17:02:22,157;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:02:22,161;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT (1) AS `a` FROM `auth_group` WHERE `auth_group`.`name` = 'group1' LIMIT 1; args=(u'group1',)
2018-09-02 17:02:22,161;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `auth_group` (`name`) VALUES ('group1'); args=[u'group1']
2018-09-02 17:02:22,165;DEBUG;utils.execute;Line:91;Message:(0.002) SELECT `auth_permission`.`id` FROM `auth_permission` INNER JOIN `auth_group_permissions` ON (`auth_permission`.`id` = `auth_group_permissions`.`permission_id`) INNER JOIN `django_content_type` ON (`auth_permission`.`content_type_id` = `django_content_type`.`id`) WHERE `auth_group_permissions`.`group_id` = 1 ORDER BY `django_content_type`.`app_label` ASC, `django_content_type`.`model` ASC, `auth_permission`.`codename` ASC; args=(1,)
2018-09-02 17:02:22,167;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `django_admin_log` (`action_time`, `user_id`, `content_type_id`, `object_id`, `object_repr`, `action_flag`, `change_message`) VALUES ('2018-09-02 05:02:22.167000', 1, 2, '1', 'group1', 1, '[{\"added\": {}}]'); args=[u'2018-09-02 05:02:22.167000', 1, 2, u'1', u'group1', 1, '[{"added": {}}]']
2018-09-02 17:02:22,207;DEBUG;utils.execute;Line:91;Message:(0.001) UPDATE `django_session` SET `session_data` = 'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', `expire_date` = '2018-09-16 05:02:22.206000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', u'2018-09-16 05:02:22.206000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:02:22,209;INFO;basehttp.log_message;Line:124;Message:"POST /admin/auth/group/add/ HTTP/1.1" 302 0
2018-09-02 17:02:22,213;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:02:22,213;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:02:22,214;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:02:22.211000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:02:22.211000')
2018-09-02 17:02:22,217;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:02:22,217;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT COUNT(*) AS `__count` FROM `auth_group`; args=()
2018-09-02 17:02:22,217;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT COUNT(*) AS `__count` FROM `auth_group`; args=()
2018-09-02 17:02:22,220;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_group`.`id`, `auth_group`.`name` FROM `auth_group` ORDER BY `auth_group`.`name` ASC, `auth_group`.`id` DESC; args=()
2018-09-02 17:02:22,230;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'errors' in template 'admin/change_list.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [errors] in u'None'
2018-09-02 17:02:22,234;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'show' in template 'admin/change_list.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [show] in u"[{'False': False, 'None': None, 'True': True, 'csrf_token': <SimpleLazyObject: <function _get_val at 0x0000000004ECE438>>}]"
2018-09-02 17:02:22,234;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'is_multipart' in template 'admin/change_list.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [is_multipart] in u'None'
2018-09-02 17:02:22,246;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'non_field_errors' in template 'admin/change_list.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [non_field_errors] in u'None'
2018-09-02 17:02:22,250;DEBUG;utils.execute;Line:91;Message:(0.001) UPDATE `django_session` SET `session_data` = 'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', `expire_date` = '2018-09-16 05:02:22.249000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', u'2018-09-16 05:02:22.249000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:02:22,253;INFO;basehttp.log_message;Line:124;Message:"GET /admin/auth/group/ HTTP/1.1" 200 4681
2018-09-02 17:02:22,312;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:02:22,312;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:02:22,315;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:02:22.311000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:02:22.311000')
2018-09-02 17:02:22,315;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:02:22,319;DEBUG;utils.execute;Line:91;Message:(0.001) UPDATE `django_session` SET `session_data` = 'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', `expire_date` = '2018-09-16 05:02:22.318000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', u'2018-09-16 05:02:22.318000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:02:22,321;INFO;basehttp.log_message;Line:124;Message:"GET /admin/jsi18n/ HTTP/1.1" 200 3217
2018-09-02 17:02:29,336;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:02:29,338;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:02:29,341;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:02:29.331000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:02:29.331000')
2018-09-02 17:02:29,342;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:02:29,351;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'is_popup' in template 'admin/app_index.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [is_popup] in u"[{'False': False, 'None': None, 'True': True}, {u'csrf_token': <SimpleLazyObject: <function _get_val at 0x0000000004FECB38>>, 'user': <SimpleLazyObject: <User: admin>>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x0000000005015278>, 'DEFAULT_MESSAGE_LEVELS': {'DEBUG': 10, 'INFO': 20, 'WARNING': 30, 'SUCCESS': 25, 'ERROR': 40}, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x0000000005015240>, u'request': <WSGIRequest: GET '/admin/auth/'>}, {}, {'available_apps': [{'app_url': u'/admin/auth/', 'models': [{'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/group/', 'object_name': 'Group', 'name': u'Groups', 'add_url': u'/admin/auth/group/add/'}, {'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/user/', 'object_name': 'User', 'name': u'Users', 'add_url': u'/admin/auth/user/add/'}], 'has_module_perms': True, 'name': u'Authentication and Authorization', 'app_label': 'auth'}], 'app_list': [{'app_url': u'/admin/auth/', 'models': [{'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/group/', 'object_name': 'Group', 'name': u'Groups', 'add_url': u'/admin/auth/group/add/'}, {'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/user/', 'object_name': 'User', 'name': u'Users', 'add_url': u'/admin/auth/user/add/'}], 'has_module_perms': True, 'name': u'Authentication and Authorization', 'app_label': 'auth'}], 'title': u'Authentication and Authorization administration', 'site_header': u'Django administration', u'LANGUAGE_CODE': 'en-us', 'site_url': '/', 'site_title': u'Django site admin', 'app_label': u'auth', 'has_permission': True, u'LANGUAGE_BIDI': False}]"
2018-09-02 17:02:29,351;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'is_popup' in template 'admin/app_index.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [is_popup] in u"[{'False': False, 'None': None, 'True': True}, {u'csrf_token': <SimpleLazyObject: <function _get_val at 0x0000000004FECB38>>, 'user': <SimpleLazyObject: <User: admin>>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x0000000005015278>, 'DEFAULT_MESSAGE_LEVELS': {'DEBUG': 10, 'INFO': 20, 'WARNING': 30, 'SUCCESS': 25, 'ERROR': 40}, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x0000000005015240>, u'request': <WSGIRequest: GET '/admin/auth/'>}, {}, {'available_apps': [{'app_url': u'/admin/auth/', 'models': [{'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/group/', 'object_name': 'Group', 'name': u'Groups', 'add_url': u'/admin/auth/group/add/'}, {'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/user/', 'object_name': 'User', 'name': u'Users', 'add_url': u'/admin/auth/user/add/'}], 'has_module_perms': True, 'name': u'Authentication and Authorization', 'app_label': 'auth'}], 'app_list': [{'app_url': u'/admin/auth/', 'models': [{'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/group/', 'object_name': 'Group', 'name': u'Groups', 'add_url': u'/admin/auth/group/add/'}, {'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/user/', 'object_name': 'User', 'name': u'Users', 'add_url': u'/admin/auth/user/add/'}], 'has_module_perms': True, 'name': u'Authentication and Authorization', 'app_label': 'auth'}], 'title': u'Authentication and Authorization administration', 'site_header': u'Django administration', u'LANGUAGE_CODE': 'en-us', 'site_url': '/', 'site_title': u'Django site admin', 'app_label': u'auth', 'has_permission': True, u'LANGUAGE_BIDI': False}]"
2018-09-02 17:02:29,357;DEBUG;utils.execute;Line:91;Message:(0.001) UPDATE `django_session` SET `session_data` = 'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', `expire_date` = '2018-09-16 05:02:29.356000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', u'2018-09-16 05:02:29.356000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:02:29,371;INFO;basehttp.log_message;Line:124;Message:"GET /admin/auth/ HTTP/1.1" 200 2705
2018-09-02 17:02:31,661;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:02:31,661;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:02:31,665;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:02:31.658000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:02:31.658000')
2018-09-02 17:02:31,667;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:02:31,667;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_group`.`id`, `auth_group`.`name` FROM `auth_group`; args=()
2018-09-02 17:02:31,671;DEBUG;utils.execute;Line:91;Message:(0.002) SELECT COUNT(*) AS `__count` FROM `auth_user`; args=()
2018-09-02 17:02:31,671;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT COUNT(*) AS `__count` FROM `auth_user`; args=()
2018-09-02 17:02:31,673;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` ORDER BY `auth_user`.`username` ASC, `auth_user`.`id` DESC; args=()
2018-09-02 17:02:31,683;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'errors' in template 'admin/change_list.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [errors] in u'None'
2018-09-02 17:02:31,687;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'show' in template 'admin/change_list.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [show] in u"[{'False': False, 'None': None, 'True': True, 'csrf_token': <SimpleLazyObject: <function _get_val at 0x0000000004FECEB8>>}]"
2018-09-02 17:02:31,693;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'is_multipart' in template 'admin/change_list.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [is_multipart] in u'None'
2018-09-02 17:02:31,704;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'non_field_errors' in template 'admin/change_list.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [non_field_errors] in u'None'
2018-09-02 17:02:31,707;DEBUG;utils.execute;Line:91;Message:(0.001) UPDATE `django_session` SET `session_data` = 'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', `expire_date` = '2018-09-16 05:02:31.707000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', u'2018-09-16 05:02:31.707000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:02:31,723;INFO;basehttp.log_message;Line:124;Message:"GET /admin/auth/user/ HTTP/1.1" 200 6703
2018-09-02 17:02:31,755;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:02:31,755;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:02:31,760;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:02:31.750000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:02:31.750000')
2018-09-02 17:02:31,763;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:02:31,766;DEBUG;utils.execute;Line:91;Message:(0.002) UPDATE `django_session` SET `session_data` = 'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', `expire_date` = '2018-09-16 05:02:31.764000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', u'2018-09-16 05:02:31.764000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:02:31,770;INFO;basehttp.log_message;Line:124;Message:"GET /admin/jsi18n/ HTTP/1.1" 200 3217
2018-09-02 17:02:39,306;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:02:39,308;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:02:39,312;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:02:39.305000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:02:39.305000')
2018-09-02 17:02:39,315;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:02:39,315;DEBUG;utils.execute;Line:91;Message:(0.000) SAVEPOINT `s220_x1`; args=None
2018-09-02 17:02:39,319;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE (`django_content_type`.`model` = 'user' AND `django_content_type`.`app_label` = 'auth'); args=('user', 'auth')
2018-09-02 17:02:39,319;DEBUG;utils.execute;Line:91;Message:(0.000) RELEASE SAVEPOINT `s220_x1`; args=None
2018-09-02 17:02:39,348;DEBUG;utils.execute;Line:91;Message:(0.001) UPDATE `django_session` SET `session_data` = 'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', `expire_date` = '2018-09-16 05:02:39.346000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', u'2018-09-16 05:02:39.346000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:02:39,405;INFO;basehttp.log_message;Line:124;Message:"GET /admin/auth/user/add/ HTTP/1.1" 200 5501
2018-09-02 17:02:39,438;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:02:39,438;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:02:39,446;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:02:39.435000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:02:39.435000')
2018-09-02 17:02:39,447;DEBUG;utils.execute;Line:91;Message:(0.002) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:02:39,450;DEBUG;utils.execute;Line:91;Message:(0.001) UPDATE `django_session` SET `session_data` = 'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', `expire_date` = '2018-09-16 05:02:39.450000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', u'2018-09-16 05:02:39.450000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:02:39,457;INFO;basehttp.log_message;Line:124;Message:"GET /admin/jsi18n/ HTTP/1.1" 200 3217
2018-09-02 17:03:00,584;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:03:00,585;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT VERSION(); args=None
2018-09-02 17:03:00,586;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:03:00.580000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:03:00.580000')
2018-09-02 17:03:00,588;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:03:00,588;DEBUG;utils.execute;Line:91;Message:(0.000) SAVEPOINT `s20344_x1`; args=None
2018-09-02 17:03:00,592;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT (1) AS `a` FROM `auth_user` WHERE `auth_user`.`username` = 'user1' LIMIT 1; args=(u'user1',)
2018-09-02 17:03:00,625;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `auth_user` (`password`, `last_login`, `is_superuser`, `username`, `first_name`, `last_name`, `email`, `is_staff`, `is_active`, `date_joined`, `role`) VALUES ('pbkdf2_sha256$36000$uQlIsYAyc4q1$zQn4/rXIxH7K9Y5FA8TRIyCgb7+3qTumBSdNeuxE/80=', NULL, 0, 'user1', '', '', '', 0, 1, '2018-09-02 05:03:00.591000', NULL); args=[u'pbkdf2_sha256$36000$uQlIsYAyc4q1$zQn4/rXIxH7K9Y5FA8TRIyCgb7+3qTumBSdNeuxE/80=', None, False, u'user1', u'', u'', u'', False, True, u'2018-09-02 05:03:00.591000', None]
2018-09-02 17:03:00,625;DEBUG;utils.execute;Line:91;Message:(0.000) INSERT INTO `django_admin_log` (`action_time`, `user_id`, `content_type_id`, `object_id`, `object_repr`, `action_flag`, `change_message`) VALUES ('2018-09-02 05:03:00.626000', 1, 4, '2', 'user1', 1, '[{\"added\": {}}]'); args=[u'2018-09-02 05:03:00.626000', 1, 4, u'2', u'user1', 1, '[{"added": {}}]']
2018-09-02 17:03:00,627;DEBUG;utils.execute;Line:91;Message:(0.000) RELEASE SAVEPOINT `s20344_x1`; args=None
2018-09-02 17:03:00,630;DEBUG;utils.execute;Line:91;Message:(0.000) UPDATE `django_session` SET `session_data` = 'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', `expire_date` = '2018-09-16 05:03:00.629000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', u'2018-09-16 05:03:00.629000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:03:00,631;INFO;basehttp.log_message;Line:124;Message:"POST /admin/auth/user/add/ HTTP/1.1" 302 0
2018-09-02 17:03:00,641;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:03:00,641;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:03:00,644;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:03:00.640000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:03:00.640000')
2018-09-02 17:03:00,644;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:03:00,645;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 2; args=(2,)
2018-09-02 17:03:00,690;DEBUG;utils.execute;Line:91;Message:(0.002) SELECT `auth_group`.`id`, `auth_group`.`name` FROM `auth_group` INNER JOIN `auth_user_groups` ON (`auth_group`.`id` = `auth_user_groups`.`group_id`) WHERE `auth_user_groups`.`user_id` = 2; args=(2,)
2018-09-02 17:03:00,690;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_group`.`id`, `auth_group`.`name` FROM `auth_group` ORDER BY `auth_group`.`name` ASC; args=()
2018-09-02 17:03:00,696;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'can_change_related' in template 'admin/widgets/related_widget_wrapper.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [can_change_related] in u'[{\'False\': False, \'None\': None, \'True\': True}, {u\'url_params\': u\'_to_field=id&_popup=1\', \'add_related_url\': u\'/admin/auth/group/add/\', u\'name\': \'groups\', \'can_add_related\': True, u\'model\': u\'group\', u\'rendered_widget\': u\'<select name="groups" data-field-name="groups" multiple="multiple" id="id_groups" data-is-stacked="0" class="selectfilter">\\n  <option value="1">group1</option>\\n\\n</select>\'}, {\'block\': <Block Node: links. Contents: [<TextNode: u\'\\n        \'>, <django.template.defaulttags.SpacelessNode object at 0x0000000005243CC0>, <TextNode: u\'\\n    \'>]>}]'
2018-09-02 17:03:00,697;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'can_delete_related' in template 'admin/widgets/related_widget_wrapper.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [can_delete_related] in u'[{\'False\': False, \'None\': None, \'True\': True}, {u\'url_params\': u\'_to_field=id&_popup=1\', \'add_related_url\': u\'/admin/auth/group/add/\', u\'name\': \'groups\', \'can_add_related\': True, u\'model\': u\'group\', u\'rendered_widget\': u\'<select name="groups" data-field-name="groups" multiple="multiple" id="id_groups" data-is-stacked="0" class="selectfilter">\\n  <option value="1">group1</option>\\n\\n</select>\'}, {\'block\': <Block Node: links. Contents: [<TextNode: u\'\\n        \'>, <django.template.defaulttags.SpacelessNode object at 0x0000000005243CC0>, <TextNode: u\'\\n    \'>]>}]'
2018-09-02 17:03:00,700;DEBUG;utils.execute;Line:91;Message:(0.002) SELECT `auth_permission`.`id`, `auth_permission`.`name`, `auth_permission`.`content_type_id`, `auth_permission`.`codename` FROM `auth_permission` INNER JOIN `auth_user_user_permissions` ON (`auth_permission`.`id` = `auth_user_user_permissions`.`permission_id`) INNER JOIN `django_content_type` ON (`auth_permission`.`content_type_id` = `django_content_type`.`id`) WHERE `auth_user_user_permissions`.`user_id` = 2 ORDER BY `django_content_type`.`app_label` ASC, `django_content_type`.`model` ASC, `auth_permission`.`codename` ASC; args=(2,)
2018-09-02 17:03:00,701;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_permission`.`id`, `auth_permission`.`name`, `auth_permission`.`content_type_id`, `auth_permission`.`codename`, `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `auth_permission` INNER JOIN `django_content_type` ON (`auth_permission`.`content_type_id` = `django_content_type`.`id`) ORDER BY `django_content_type`.`app_label` ASC, `django_content_type`.`model` ASC, `auth_permission`.`codename` ASC; args=()
2018-09-02 17:03:00,710;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'can_change_related' in template 'admin/widgets/related_widget_wrapper.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [can_change_related] in u'[{\'False\': False, \'None\': None, \'True\': True}, {u\'model\': u\'permission\', u\'url_params\': u\'_to_field=id&_popup=1\', u\'rendered_widget\': u\'<select name="user_permissions" data-field-name="user permissions" multiple="multiple" id="id_user_permissions" data-is-stacked="0" class="selectfilter">\\n  <option value="1">admin | log entry | Can add log entry</option>\\n\\n  <option value="2">admin | log entry | Can change log entry</option>\\n\\n  <option value="3">admin | log entry | Can delete log entry</option>\\n\\n  <option value="4">auth | group | Can add group</option>\\n\\n  <option value="5">auth | group | Can change group</option>\\n\\n  <option value="6">auth | group | Can delete group</option>\\n\\n  <option value="7">auth | permission | Can add permission</option>\\n\\n  <option value="8">auth | permission | Can change permission</option>\\n\\n  <option value="9">auth | permission | Can delete permission</option>\\n\\n  <option value="10">auth | user | Can add user</option>\\n\\n  <option value="11">auth | user | Can change user</option>\\n\\n  <option value="12">auth | user | Can delete user</option>\\n\\n  <option value="13">contenttypes | content type | Can add content type</option>\\n\\n  <option value="14">contenttypes | content type | Can change content type</option>\\n\\n  <option value="15">contenttypes | content type | Can delete content type</option>\\n\\n  <option value="16">sessions | session | Can add session</option>\\n\\n  <option value="17">sessions | session | Can change session</option>\\n\\n  <option value="18">sessions | session | Can delete session</option>\\n\\n</select>\', u\'name\': \'user_permissions\'}, {\'block\': <Block Node: links. Contents: [<TextNode: u\'\\n        \'>, <django.template.defaulttags.SpacelessNode object at 0x000000000523A438>, <TextNode: u\'\\n    \'>]>}]'
2018-09-02 17:03:00,710;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'can_add_related' in template 'admin/widgets/related_widget_wrapper.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [can_add_related] in u'[{\'False\': False, \'None\': None, \'True\': True}, {u\'model\': u\'permission\', u\'url_params\': u\'_to_field=id&_popup=1\', u\'rendered_widget\': u\'<select name="user_permissions" data-field-name="user permissions" multiple="multiple" id="id_user_permissions" data-is-stacked="0" class="selectfilter">\\n  <option value="1">admin | log entry | Can add log entry</option>\\n\\n  <option value="2">admin | log entry | Can change log entry</option>\\n\\n  <option value="3">admin | log entry | Can delete log entry</option>\\n\\n  <option value="4">auth | group | Can add group</option>\\n\\n  <option value="5">auth | group | Can change group</option>\\n\\n  <option value="6">auth | group | Can delete group</option>\\n\\n  <option value="7">auth | permission | Can add permission</option>\\n\\n  <option value="8">auth | permission | Can change permission</option>\\n\\n  <option value="9">auth | permission | Can delete permission</option>\\n\\n  <option value="10">auth | user | Can add user</option>\\n\\n  <option value="11">auth | user | Can change user</option>\\n\\n  <option value="12">auth | user | Can delete user</option>\\n\\n  <option value="13">contenttypes | content type | Can add content type</option>\\n\\n  <option value="14">contenttypes | content type | Can change content type</option>\\n\\n  <option value="15">contenttypes | content type | Can delete content type</option>\\n\\n  <option value="16">sessions | session | Can add session</option>\\n\\n  <option value="17">sessions | session | Can change session</option>\\n\\n  <option value="18">sessions | session | Can delete session</option>\\n\\n</select>\', u\'name\': \'user_permissions\'}, {\'block\': <Block Node: links. Contents: [<TextNode: u\'\\n        \'>, <django.template.defaulttags.SpacelessNode object at 0x000000000523A438>, <TextNode: u\'\\n    \'>]>}]'
2018-09-02 17:03:00,710;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'can_delete_related' in template 'admin/widgets/related_widget_wrapper.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [can_delete_related] in u'[{\'False\': False, \'None\': None, \'True\': True}, {u\'model\': u\'permission\', u\'url_params\': u\'_to_field=id&_popup=1\', u\'rendered_widget\': u\'<select name="user_permissions" data-field-name="user permissions" multiple="multiple" id="id_user_permissions" data-is-stacked="0" class="selectfilter">\\n  <option value="1">admin | log entry | Can add log entry</option>\\n\\n  <option value="2">admin | log entry | Can change log entry</option>\\n\\n  <option value="3">admin | log entry | Can delete log entry</option>\\n\\n  <option value="4">auth | group | Can add group</option>\\n\\n  <option value="5">auth | group | Can change group</option>\\n\\n  <option value="6">auth | group | Can delete group</option>\\n\\n  <option value="7">auth | permission | Can add permission</option>\\n\\n  <option value="8">auth | permission | Can change permission</option>\\n\\n  <option value="9">auth | permission | Can delete permission</option>\\n\\n  <option value="10">auth | user | Can add user</option>\\n\\n  <option value="11">auth | user | Can change user</option>\\n\\n  <option value="12">auth | user | Can delete user</option>\\n\\n  <option value="13">contenttypes | content type | Can add content type</option>\\n\\n  <option value="14">contenttypes | content type | Can change content type</option>\\n\\n  <option value="15">contenttypes | content type | Can delete content type</option>\\n\\n  <option value="16">sessions | session | Can add session</option>\\n\\n  <option value="17">sessions | session | Can change session</option>\\n\\n  <option value="18">sessions | session | Can delete session</option>\\n\\n</select>\', u\'name\': \'user_permissions\'}, {\'block\': <Block Node: links. Contents: [<TextNode: u\'\\n        \'>, <django.template.defaulttags.SpacelessNode object at 0x000000000523A438>, <TextNode: u\'\\n    \'>]>}]'
2018-09-02 17:03:00,747;DEBUG;utils.execute;Line:91;Message:(0.001) UPDATE `django_session` SET `session_data` = 'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', `expire_date` = '2018-09-16 05:03:00.746000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', u'2018-09-16 05:03:00.746000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:03:00,749;INFO;basehttp.log_message;Line:124;Message:"GET /admin/auth/user/2/change/ HTTP/1.1" 200 14356
2018-09-02 17:03:00,838;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:03:00,839;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:03:00,841;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:03:00.838000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:03:00.838000')
2018-09-02 17:03:00,842;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:03:00,845;DEBUG;utils.execute;Line:91;Message:(0.000) UPDATE `django_session` SET `session_data` = 'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', `expire_date` = '2018-09-16 05:03:00.844000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', u'2018-09-16 05:03:00.844000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:03:00,848;INFO;basehttp.log_message;Line:124;Message:"GET /admin/jsi18n/ HTTP/1.1" 200 3217
2018-09-02 17:03:00,859;INFO;basehttp.log_message;Line:124;Message:"GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
2018-09-02 17:03:01,220;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [name] in u'<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>'
2018-09-02 17:03:01,221;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [name] in u'<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>'
2018-09-02 17:03:01,226;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:03:01,226;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:03:01,227;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:03:01,227;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:03:01,232;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:03:01.224000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:03:01.224000')
2018-09-02 17:03:01,232;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:03:01.224000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:03:01.224000')
2018-09-02 17:03:01,236;DEBUG;utils.execute;Line:91;Message:(0.001) UPDATE `django_session` SET `session_data` = 'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', `expire_date` = '2018-09-16 05:03:01.233000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', u'2018-09-16 05:03:01.233000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:03:01,243;DEBUG;utils.execute;Line:91;Message:(0.008) UPDATE `django_session` SET `session_data` = 'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', `expire_date` = '2018-09-16 05:03:01.233000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', u'2018-09-16 05:03:01.233000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:03:01,243;WARNING;base.get_response;Line:152;Message:Not Found: /missing-admin-media-prefix/img/icon_calendar.gif
2018-09-02 17:03:01,244;WARNING;base.get_response;Line:152;Message:Not Found: /missing-admin-media-prefix/img/icon_clock.gif
2018-09-02 17:03:01,244;WARNING;basehttp.log_message;Line:124;Message:"GET /missing-admin-media-prefix/img/icon_calendar.gif HTTP/1.1" 404 2088
2018-09-02 17:03:01,244;WARNING;basehttp.log_message;Line:124;Message:"GET /missing-admin-media-prefix/img/icon_clock.gif HTTP/1.1" 404 2079
2018-09-02 17:04:21,098;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:04:21,098;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:04:21,101;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:04:21.095000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:04:21.095000')
2018-09-02 17:04:21,102;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:04:21,104;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 2; args=(2,)
2018-09-02 17:04:21,111;DEBUG;utils.execute;Line:91;Message:(0.002) SELECT `auth_group`.`id`, `auth_group`.`name` FROM `auth_group` WHERE `auth_group`.`id` IN (1) ORDER BY `auth_group`.`name` ASC; args=(1,)
2018-09-02 17:04:21,118;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT (1) AS `a` FROM `auth_user` WHERE (`auth_user`.`username` = 'user1' AND NOT (`auth_user`.`id` = 2)) LIMIT 1; args=(u'user1', 2)
2018-09-02 17:04:21,121;DEBUG;utils.execute;Line:91;Message:(0.001) UPDATE `auth_user` SET `password` = 'pbkdf2_sha256$36000$uQlIsYAyc4q1$zQn4/rXIxH7K9Y5FA8TRIyCgb7+3qTumBSdNeuxE/80=', `last_login` = NULL, `is_superuser` = 0, `username` = 'user1', `first_name` = 'Tong', `last_name` = 'Xing', `email` = 'x1980p@gmail.com', `is_staff` = 0, `is_active` = 1, `date_joined` = '2018-09-02 05:03:00', `role` = '1' WHERE `auth_user`.`id` = 2; args=(u'pbkdf2_sha256$36000$uQlIsYAyc4q1$zQn4/rXIxH7K9Y5FA8TRIyCgb7+3qTumBSdNeuxE/80=', False, u'user1', u'Tong', u'Xing', u'x1980p@gmail.com', False, True, u'2018-09-02 05:03:00', u'1', 2)
2018-09-02 17:04:21,124;DEBUG;utils.execute;Line:91;Message:(0.002) SELECT `auth_group`.`id` FROM `auth_group` INNER JOIN `auth_user_groups` ON (`auth_group`.`id` = `auth_user_groups`.`group_id`) WHERE `auth_user_groups`.`user_id` = 2; args=(2,)
2018-09-02 17:04:21,125;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_user_groups`.`group_id` FROM `auth_user_groups` WHERE (`auth_user_groups`.`user_id` = 2 AND `auth_user_groups`.`group_id` IN (1)); args=(2, 1)
2018-09-02 17:04:21,128;DEBUG;utils.execute;Line:91;Message:(0.001) INSERT INTO `auth_user_groups` (`user_id`, `group_id`) VALUES (2, 1); args=(2, 1)
2018-09-02 17:04:21,131;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `auth_permission`.`id` FROM `auth_permission` INNER JOIN `auth_user_user_permissions` ON (`auth_permission`.`id` = `auth_user_user_permissions`.`permission_id`) INNER JOIN `django_content_type` ON (`auth_permission`.`content_type_id` = `django_content_type`.`id`) WHERE `auth_user_user_permissions`.`user_id` = 2 ORDER BY `django_content_type`.`app_label` ASC, `django_content_type`.`model` ASC, `auth_permission`.`codename` ASC; args=(2,)
2018-09-02 17:04:21,134;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_group`.`id`, `auth_group`.`name` FROM `auth_group` INNER JOIN `auth_user_groups` ON (`auth_group`.`id` = `auth_user_groups`.`group_id`) WHERE `auth_user_groups`.`user_id` = 2; args=(2,)
2018-09-02 17:04:21,135;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `auth_permission`.`id`, `auth_permission`.`name`, `auth_permission`.`content_type_id`, `auth_permission`.`codename` FROM `auth_permission` INNER JOIN `auth_user_user_permissions` ON (`auth_permission`.`id` = `auth_user_user_permissions`.`permission_id`) INNER JOIN `django_content_type` ON (`auth_permission`.`content_type_id` = `django_content_type`.`id`) WHERE `auth_user_user_permissions`.`user_id` = 2 ORDER BY `django_content_type`.`app_label` ASC, `django_content_type`.`model` ASC, `auth_permission`.`codename` ASC; args=(2,)
2018-09-02 17:04:21,138;DEBUG;utils.execute;Line:91;Message:(0.002) INSERT INTO `django_admin_log` (`action_time`, `user_id`, `content_type_id`, `object_id`, `object_repr`, `action_flag`, `change_message`) VALUES ('2018-09-02 05:04:21.136000', 1, 4, '2', 'user1', 2, '[{\"changed\": {\"fields\": [\"first_name\", \"last_name\", \"email\", \"role\"]}}]'); args=[u'2018-09-02 05:04:21.136000', 1, 4, u'2', u'user1', 2, '[{"changed": {"fields": ["first_name", "last_name", "email", "role"]}}]']
2018-09-02 17:04:21,188;DEBUG;utils.execute;Line:91;Message:(0.000) UPDATE `django_session` SET `session_data` = 'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', `expire_date` = '2018-09-16 05:04:21.188000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', u'2018-09-16 05:04:21.188000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:04:21,190;INFO;basehttp.log_message;Line:124;Message:"POST /admin/auth/user/2/change/ HTTP/1.1" 302 0
2018-09-02 17:04:21,196;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:04:21,197;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:04:21,198;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:04:21.194000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:04:21.194000')
2018-09-02 17:04:21,200;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:04:21,200;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_group`.`id`, `auth_group`.`name` FROM `auth_group`; args=()
2018-09-02 17:04:21,203;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT COUNT(*) AS `__count` FROM `auth_user`; args=()
2018-09-02 17:04:21,203;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT COUNT(*) AS `__count` FROM `auth_user`; args=()
2018-09-02 17:04:21,206;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` ORDER BY `auth_user`.`username` ASC, `auth_user`.`id` DESC; args=()
2018-09-02 17:04:21,217;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'errors' in template 'admin/change_list.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [errors] in u'None'
2018-09-02 17:04:21,220;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'show' in template 'admin/change_list.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [show] in u"[{'False': False, 'None': None, 'True': True, 'csrf_token': <SimpleLazyObject: <function _get_val at 0x0000000004E5F3C8>>}]"
2018-09-02 17:04:21,224;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'is_multipart' in template 'admin/change_list.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [is_multipart] in u'None'
2018-09-02 17:04:21,239;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'non_field_errors' in template 'admin/change_list.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [non_field_errors] in u'None'
2018-09-02 17:04:21,239;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'non_field_errors' in template 'admin/change_list.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [non_field_errors] in u'None'
2018-09-02 17:04:21,243;DEBUG;utils.execute;Line:91;Message:(0.001) UPDATE `django_session` SET `session_data` = 'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', `expire_date` = '2018-09-16 05:04:21.242000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', u'2018-09-16 05:04:21.242000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:04:21,250;INFO;basehttp.log_message;Line:124;Message:"GET /admin/auth/user/ HTTP/1.1" 200 7303
2018-09-02 17:04:21,308;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:04:21,309;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:04:21,311;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:04:21.307000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:04:21.307000')
2018-09-02 17:04:21,311;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:04:21,312;DEBUG;utils.execute;Line:91;Message:(0.000) UPDATE `django_session` SET `session_data` = 'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', `expire_date` = '2018-09-16 05:04:21.312000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', u'2018-09-16 05:04:21.312000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:04:21,315;INFO;basehttp.log_message;Line:124;Message:"GET /admin/jsi18n/ HTTP/1.1" 200 3217
2018-09-02 17:04:21,499;INFO;basehttp.log_message;Line:124;Message:"GET /static/admin/img/icon-no.svg HTTP/1.1" 200 560
2018-09-02 17:04:30,299;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:04:30,301;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:04:30,303;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:04:30.296000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:04:30.296000')
2018-09-02 17:04:30,305;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:04:30,308;DEBUG;utils.execute;Line:91;Message:(0.002) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 2; args=(2,)
2018-09-02 17:04:30,348;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_group`.`id`, `auth_group`.`name` FROM `auth_group` INNER JOIN `auth_user_groups` ON (`auth_group`.`id` = `auth_user_groups`.`group_id`) WHERE `auth_user_groups`.`user_id` = 2; args=(2,)
2018-09-02 17:04:30,349;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_group`.`id`, `auth_group`.`name` FROM `auth_group` ORDER BY `auth_group`.`name` ASC; args=()
2018-09-02 17:04:30,355;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'can_change_related' in template 'admin/widgets/related_widget_wrapper.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [can_change_related] in u'[{\'False\': False, \'None\': None, \'True\': True}, {u\'url_params\': u\'_to_field=id&_popup=1\', \'add_related_url\': u\'/admin/auth/group/add/\', u\'name\': \'groups\', \'can_add_related\': True, u\'model\': u\'group\', u\'rendered_widget\': u\'<select name="groups" data-field-name="groups" multiple="multiple" id="id_groups" data-is-stacked="0" class="selectfilter">\\n  <option value="1" selected>group1</option>\\n\\n</select>\'}, {\'block\': <Block Node: links. Contents: [<TextNode: u\'\\n        \'>, <django.template.defaulttags.SpacelessNode object at 0x00000000052567B8>, <TextNode: u\'\\n    \'>]>}]'
2018-09-02 17:04:30,357;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'can_delete_related' in template 'admin/widgets/related_widget_wrapper.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [can_delete_related] in u'[{\'False\': False, \'None\': None, \'True\': True}, {u\'url_params\': u\'_to_field=id&_popup=1\', \'add_related_url\': u\'/admin/auth/group/add/\', u\'name\': \'groups\', \'can_add_related\': True, u\'model\': u\'group\', u\'rendered_widget\': u\'<select name="groups" data-field-name="groups" multiple="multiple" id="id_groups" data-is-stacked="0" class="selectfilter">\\n  <option value="1" selected>group1</option>\\n\\n</select>\'}, {\'block\': <Block Node: links. Contents: [<TextNode: u\'\\n        \'>, <django.template.defaulttags.SpacelessNode object at 0x00000000052567B8>, <TextNode: u\'\\n    \'>]>}]'
2018-09-02 17:04:30,361;DEBUG;utils.execute;Line:91;Message:(0.002) SELECT `auth_permission`.`id`, `auth_permission`.`name`, `auth_permission`.`content_type_id`, `auth_permission`.`codename` FROM `auth_permission` INNER JOIN `auth_user_user_permissions` ON (`auth_permission`.`id` = `auth_user_user_permissions`.`permission_id`) INNER JOIN `django_content_type` ON (`auth_permission`.`content_type_id` = `django_content_type`.`id`) WHERE `auth_user_user_permissions`.`user_id` = 2 ORDER BY `django_content_type`.`app_label` ASC, `django_content_type`.`model` ASC, `auth_permission`.`codename` ASC; args=(2,)
2018-09-02 17:04:30,364;DEBUG;utils.execute;Line:91;Message:(0.002) SELECT `auth_permission`.`id`, `auth_permission`.`name`, `auth_permission`.`content_type_id`, `auth_permission`.`codename`, `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `auth_permission` INNER JOIN `django_content_type` ON (`auth_permission`.`content_type_id` = `django_content_type`.`id`) ORDER BY `django_content_type`.`app_label` ASC, `django_content_type`.`model` ASC, `auth_permission`.`codename` ASC; args=()
2018-09-02 17:04:30,371;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'can_change_related' in template 'admin/widgets/related_widget_wrapper.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [can_change_related] in u'[{\'False\': False, \'None\': None, \'True\': True}, {u\'model\': u\'permission\', u\'url_params\': u\'_to_field=id&_popup=1\', u\'rendered_widget\': u\'<select name="user_permissions" data-field-name="user permissions" multiple="multiple" id="id_user_permissions" data-is-stacked="0" class="selectfilter">\\n  <option value="1">admin | log entry | Can add log entry</option>\\n\\n  <option value="2">admin | log entry | Can change log entry</option>\\n\\n  <option value="3">admin | log entry | Can delete log entry</option>\\n\\n  <option value="4">auth | group | Can add group</option>\\n\\n  <option value="5">auth | group | Can change group</option>\\n\\n  <option value="6">auth | group | Can delete group</option>\\n\\n  <option value="7">auth | permission | Can add permission</option>\\n\\n  <option value="8">auth | permission | Can change permission</option>\\n\\n  <option value="9">auth | permission | Can delete permission</option>\\n\\n  <option value="10">auth | user | Can add user</option>\\n\\n  <option value="11">auth | user | Can change user</option>\\n\\n  <option value="12">auth | user | Can delete user</option>\\n\\n  <option value="13">contenttypes | content type | Can add content type</option>\\n\\n  <option value="14">contenttypes | content type | Can change content type</option>\\n\\n  <option value="15">contenttypes | content type | Can delete content type</option>\\n\\n  <option value="16">sessions | session | Can add session</option>\\n\\n  <option value="17">sessions | session | Can change session</option>\\n\\n  <option value="18">sessions | session | Can delete session</option>\\n\\n</select>\', u\'name\': \'user_permissions\'}, {\'block\': <Block Node: links. Contents: [<TextNode: u\'\\n        \'>, <django.template.defaulttags.SpacelessNode object at 0x00000000052631D0>, <TextNode: u\'\\n    \'>]>}]'
2018-09-02 17:04:30,371;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'can_add_related' in template 'admin/widgets/related_widget_wrapper.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [can_add_related] in u'[{\'False\': False, \'None\': None, \'True\': True}, {u\'model\': u\'permission\', u\'url_params\': u\'_to_field=id&_popup=1\', u\'rendered_widget\': u\'<select name="user_permissions" data-field-name="user permissions" multiple="multiple" id="id_user_permissions" data-is-stacked="0" class="selectfilter">\\n  <option value="1">admin | log entry | Can add log entry</option>\\n\\n  <option value="2">admin | log entry | Can change log entry</option>\\n\\n  <option value="3">admin | log entry | Can delete log entry</option>\\n\\n  <option value="4">auth | group | Can add group</option>\\n\\n  <option value="5">auth | group | Can change group</option>\\n\\n  <option value="6">auth | group | Can delete group</option>\\n\\n  <option value="7">auth | permission | Can add permission</option>\\n\\n  <option value="8">auth | permission | Can change permission</option>\\n\\n  <option value="9">auth | permission | Can delete permission</option>\\n\\n  <option value="10">auth | user | Can add user</option>\\n\\n  <option value="11">auth | user | Can change user</option>\\n\\n  <option value="12">auth | user | Can delete user</option>\\n\\n  <option value="13">contenttypes | content type | Can add content type</option>\\n\\n  <option value="14">contenttypes | content type | Can change content type</option>\\n\\n  <option value="15">contenttypes | content type | Can delete content type</option>\\n\\n  <option value="16">sessions | session | Can add session</option>\\n\\n  <option value="17">sessions | session | Can change session</option>\\n\\n  <option value="18">sessions | session | Can delete session</option>\\n\\n</select>\', u\'name\': \'user_permissions\'}, {\'block\': <Block Node: links. Contents: [<TextNode: u\'\\n        \'>, <django.template.defaulttags.SpacelessNode object at 0x00000000052631D0>, <TextNode: u\'\\n    \'>]>}]'
2018-09-02 17:04:30,371;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'can_delete_related' in template 'admin/widgets/related_widget_wrapper.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [can_delete_related] in u'[{\'False\': False, \'None\': None, \'True\': True}, {u\'model\': u\'permission\', u\'url_params\': u\'_to_field=id&_popup=1\', u\'rendered_widget\': u\'<select name="user_permissions" data-field-name="user permissions" multiple="multiple" id="id_user_permissions" data-is-stacked="0" class="selectfilter">\\n  <option value="1">admin | log entry | Can add log entry</option>\\n\\n  <option value="2">admin | log entry | Can change log entry</option>\\n\\n  <option value="3">admin | log entry | Can delete log entry</option>\\n\\n  <option value="4">auth | group | Can add group</option>\\n\\n  <option value="5">auth | group | Can change group</option>\\n\\n  <option value="6">auth | group | Can delete group</option>\\n\\n  <option value="7">auth | permission | Can add permission</option>\\n\\n  <option value="8">auth | permission | Can change permission</option>\\n\\n  <option value="9">auth | permission | Can delete permission</option>\\n\\n  <option value="10">auth | user | Can add user</option>\\n\\n  <option value="11">auth | user | Can change user</option>\\n\\n  <option value="12">auth | user | Can delete user</option>\\n\\n  <option value="13">contenttypes | content type | Can add content type</option>\\n\\n  <option value="14">contenttypes | content type | Can change content type</option>\\n\\n  <option value="15">contenttypes | content type | Can delete content type</option>\\n\\n  <option value="16">sessions | session | Can add session</option>\\n\\n  <option value="17">sessions | session | Can change session</option>\\n\\n  <option value="18">sessions | session | Can delete session</option>\\n\\n</select>\', u\'name\': \'user_permissions\'}, {\'block\': <Block Node: links. Contents: [<TextNode: u\'\\n        \'>, <django.template.defaulttags.SpacelessNode object at 0x00000000052631D0>, <TextNode: u\'\\n    \'>]>}]'
2018-09-02 17:04:30,394;DEBUG;utils.execute;Line:91;Message:(0.001) UPDATE `django_session` SET `session_data` = 'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', `expire_date` = '2018-09-16 05:04:30.393000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', u'2018-09-16 05:04:30.393000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:04:30,400;INFO;basehttp.log_message;Line:124;Message:"GET /admin/auth/user/2/change/ HTTP/1.1" 200 14214
2018-09-02 17:04:30,431;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:04:30,431;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:04:30,434;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:04:30.429000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:04:30.429000')
2018-09-02 17:04:30,436;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:04:30,437;DEBUG;utils.execute;Line:91;Message:(0.000) UPDATE `django_session` SET `session_data` = 'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', `expire_date` = '2018-09-16 05:04:30.437000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'ZDMwMmM1MTFkNTQ1OTgzYzQzZGM0NTIyZjk0NzNmZWZhZTA0OTUyNzp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', u'2018-09-16 05:04:30.437000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:04:30,447;INFO;basehttp.log_message;Line:124;Message:"GET /admin/jsi18n/ HTTP/1.1" 200 3217
2018-09-02 17:04:30,716;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [name] in u'<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>'
2018-09-02 17:04:30,717;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [name] in u'<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>'
2018-09-02 17:04:30,723;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:04:30,723;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:04:30,723;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:04:30,723;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:04:30,729;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:04:30.719000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:04:30.719000')
2018-09-02 17:04:30,730;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:04:30.721000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:04:30.721000')
2018-09-02 17:04:30,732;DEBUG;utils.execute;Line:91;Message:(0.000) UPDATE `django_session` SET `session_data` = 'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', `expire_date` = '2018-09-16 05:04:30.731000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', u'2018-09-16 05:04:30.731000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:04:30,734;DEBUG;utils.execute;Line:91;Message:(0.003) UPDATE `django_session` SET `session_data` = 'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', `expire_date` = '2018-09-16 05:04:30.731000' WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=(u'YWJiNzlhZjU5YTc2N2IzOTQ4OWRiYThkOWNmMmZkYWY0NzIzZWMwZTp7Il9hdXRoX3VzZXJfaGFzaCI6IjdjN2NmZTViNjg1YmNiOWRhNDQ5NjdhZDg4MzU4NzViMjAxMmMwZmEiLCJfYXV0aF91c2VyX2JhY2tlbmQiOiJkamFuZ28uY29udHJpYi5hdXRoLmJhY2tlbmRzLk1vZGVsQmFja2VuZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', u'2018-09-16 05:04:30.731000', 'li4w1895i6ez8vw5iyzbs07k57o0yv8q')
2018-09-02 17:04:30,734;WARNING;base.get_response;Line:152;Message:Not Found: /missing-admin-media-prefix/img/icon_calendar.gif
2018-09-02 17:04:30,736;WARNING;basehttp.log_message;Line:124;Message:"GET /missing-admin-media-prefix/img/icon_calendar.gif HTTP/1.1" 404 2088
2018-09-02 17:04:30,736;WARNING;base.get_response;Line:152;Message:Not Found: /missing-admin-media-prefix/img/icon_clock.gif
2018-09-02 17:04:30,736;WARNING;basehttp.log_message;Line:124;Message:"GET /missing-admin-media-prefix/img/icon_clock.gif HTTP/1.1" 404 2079
2018-09-02 17:04:50,181;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:04:50,183;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:04:50,186;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q' AND `django_session`.`expire_date` > '2018-09-02 05:04:50.179000'); args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q', u'2018-09-02 05:04:50.179000')
2018-09-02 17:04:50,187;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`role` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2018-09-02 17:04:50,188;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE `django_session`.`session_key` = 'li4w1895i6ez8vw5iyzbs07k57o0yv8q'; args=('li4w1895i6ez8vw5iyzbs07k57o0yv8q',)
2018-09-02 17:04:50,190;DEBUG;utils.execute;Line:91;Message:(0.001) DELETE FROM `django_session` WHERE `django_session`.`session_key` IN ('li4w1895i6ez8vw5iyzbs07k57o0yv8q'); args=(u'li4w1895i6ez8vw5iyzbs07k57o0yv8q',)
2018-09-02 17:04:50,223;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'is_popup' in template 'registration/logged_out.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [is_popup] in u"[{'False': False, 'None': None, 'True': True}, {u'csrf_token': <SimpleLazyObject: <function _get_val at 0x0000000004FF9198>>, 'user': <django.contrib.auth.models.AnonymousUser object at 0x0000000005015588>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x0000000004FD3470>, 'DEFAULT_MESSAGE_LEVELS': {'DEBUG': 10, 'INFO': 20, 'WARNING': 30, 'SUCCESS': 25, 'ERROR': 40}, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x0000000005015400>, u'request': <WSGIRequest: GET '/admin/logout/'>}, {}, {'available_apps': [{'app_url': u'/admin/auth/', 'models': [{'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/group/', 'object_name': 'Group', 'name': u'Groups', 'add_url': u'/admin/auth/group/add/'}, {'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/user/', 'object_name': 'User', 'name': u'Users', 'add_url': u'/admin/auth/user/add/'}], 'has_module_perms': True, 'name': u'Authentication and Authorization', 'app_label': 'auth'}], 'site_name': '127.0.0.1:8000', 'title': u'Logged out', 'site_header': u'Django administration', 'site': <django.contrib.sites.requests.RequestSite object at 0x0000000005015B38>, 'site_url': '/', 'site_title': u'Django site admin', u'LANGUAGE_CODE': 'en-us', 'has_permission': False, u'LANGUAGE_BIDI': False, u'view': <django.contrib.auth.views.LogoutView object at 0x00000000050150F0>}]"
2018-09-02 17:04:50,223;DEBUG;base._resolve_lookup;Line:925;Message:Exception while resolving variable 'is_popup' in template 'registration/logged_out.html'.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\django\template\base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [is_popup] in u"[{'False': False, 'None': None, 'True': True}, {u'csrf_token': <SimpleLazyObject: <function _get_val at 0x0000000004FF9198>>, 'user': <django.contrib.auth.models.AnonymousUser object at 0x0000000005015588>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x0000000004FD3470>, 'DEFAULT_MESSAGE_LEVELS': {'DEBUG': 10, 'INFO': 20, 'WARNING': 30, 'SUCCESS': 25, 'ERROR': 40}, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x0000000005015400>, u'request': <WSGIRequest: GET '/admin/logout/'>}, {}, {'available_apps': [{'app_url': u'/admin/auth/', 'models': [{'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/group/', 'object_name': 'Group', 'name': u'Groups', 'add_url': u'/admin/auth/group/add/'}, {'perms': {u'add': True, u'change': True, u'delete': True}, 'admin_url': u'/admin/auth/user/', 'object_name': 'User', 'name': u'Users', 'add_url': u'/admin/auth/user/add/'}], 'has_module_perms': True, 'name': u'Authentication and Authorization', 'app_label': 'auth'}], 'site_name': '127.0.0.1:8000', 'title': u'Logged out', 'site_header': u'Django administration', 'site': <django.contrib.sites.requests.RequestSite object at 0x0000000005015B38>, 'site_url': '/', 'site_title': u'Django site admin', u'LANGUAGE_CODE': 'en-us', 'has_permission': False, u'LANGUAGE_BIDI': False, u'view': <django.contrib.auth.views.LogoutView object at 0x00000000050150F0>}]"
2018-09-02 17:04:50,224;INFO;basehttp.log_message;Line:124;Message:"GET /admin/logout/ HTTP/1.1" 200 1010
2018-09-02 17:06:02,648;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:06:02,648;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT VERSION(); args=None
2018-09-02 17:06:02,694;DEBUG;utils.execute;Line:91;Message:(0.004) SHOW FULL TABLES; args=None
2018-09-02 17:06:02,698;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2018-09-02 17:07:12,688;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:07:12,690;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:07:12,730;DEBUG;utils.execute;Line:91;Message:(0.001) SHOW FULL TABLES; args=None
2018-09-02 17:07:12,733;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2018-09-02 17:08:51,769;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:08:51,770;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT VERSION(); args=None
2018-09-02 17:08:51,811;DEBUG;utils.execute;Line:91;Message:(0.002) SHOW FULL TABLES; args=None
2018-09-02 17:08:51,815;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2018-09-02 17:09:41,102;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT @@SQL_AUTO_IS_NULL; args=None
2018-09-02 17:09:41,104;DEBUG;utils.execute;Line:91;Message:(0.001) SELECT VERSION(); args=None
2018-09-02 17:09:41,144;DEBUG;utils.execute;Line:91;Message:(0.001) SHOW FULL TABLES; args=None
2018-09-02 17:09:41,147;DEBUG;utils.execute;Line:91;Message:(0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
